<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>5&nbsp; Standardized Vocabularies – The Book of OHDSI 2nd Edition</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../representation/etl.html" rel="next">
<link href="../representation/cdm.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark-2fef5ea3f8957b3e4ecc936fc74692ca.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-44c0de8e9ef02dc9d80cb94bae68299f.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark-20d34efdb5b2a029ab0a8952a5054f92.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="../site_libs/bootstrap/bootstrap-44c0de8e9ef02dc9d80cb94bae68299f.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script src="../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>

      .quarto-title-block .quarto-title-banner h1,
      .quarto-title-block .quarto-title-banner h2,
      .quarto-title-block .quarto-title-banner h3,
      .quarto-title-block .quarto-title-banner h4,
      .quarto-title-block .quarto-title-banner h5,
      .quarto-title-block .quarto-title-banner h6
      {
        color: #FCB754;
      }

      .quarto-title-block .quarto-title-banner {
        color: #FCB754;
background-image: url(../assets/images/h243-ohdsi-logo-with-text.png);
background-size: cover;
      }
</style>


<link rel="stylesheet" href="../assets/font-awesome.css">
<link rel="stylesheet" href="../assets/tables.css">
<meta name="citation_title" content="[[5]{.chapter-number}&nbsp; [Standardized Vocabularies]{.chapter-title}]{#sec-representation-vocabularies .quarto-section-identifier}">
<meta name="citation_abstract" content="TODO- write abstract">
<meta name="citation_language" content="en">
<meta name="citation_reference" content="citation_title=The elements of statistical learning: Data mining, inference, and prediction;,citation_author=Trevor Hastie;,citation_author=Robert Tibshirani;,citation_author=Jerome H Friedman;,citation_author=Jerome H Friedman;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_volume=2;">
<meta name="citation_reference" content="citation_title=Hands-on machine learning with scikit-learn, keras, and TensorFlow;,citation_author=Aurélien Géron;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;">
<meta name="citation_reference" content="citation_title=Deep learning with python;,citation_author=Francois Chollet;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;">
<meta name="citation_reference" content="citation_title=17. A value for n-person games;,citation_author=L. S. Shapley;,citation_publication_date=1953-12;,citation_cover_date=1953-12;,citation_year=1953;,citation_fulltext_html_url=https://doi.org/10.1515/9781400881970-018;,citation_doi=10.1515/9781400881970-018;,citation_inbook_title=Contributions to the theory of games (AM-28), volume II;">
<meta name="citation_reference" content="citation_title=Consistent individualized feature attribution for tree ensembles;,citation_author=Scott M. Lundberg;,citation_author=Gabriel G. Erion;,citation_author=Su-In Lee;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://arxiv.org/abs/1802.03888;,citation_doi=10.48550/ARXIV.1802.03888;,citation_publisher=arXiv;">
<meta name="citation_reference" content="citation_title=Artificial intelligence: Emerging applications in biotechnology and pharma;,citation_author=David Sahner;,citation_author=David C. Spellmeyer;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1016/b978-0-12-815585-1.00028-0;,citation_doi=10.1016/b978-0-12-815585-1.00028-0;,citation_inbook_title=Biotechnology entrepreneurship;">
<meta name="citation_reference" content="citation_title=Data normalization and standardization: A technical report;,citation_author=Peshawa J Muhammad Ali;,citation_author=Rezhna Hassan Faraj;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_fulltext_html_url=http://rgdoi.net/10.13140/RG.2.2.28948.04489;,citation_doi=10.13140/RG.2.2.28948.04489;,citation_publisher=Unpublished;">
<meta name="citation_reference" content="citation_title=A method for comparing multiple imputation techniques: A case study on the u.s. National COVID cohort collaborative;,citation_author=Elena Casiraghi;,citation_author=Rachel Wong;,citation_author=Margaret Hall;,citation_author=Ben Coleman;,citation_author=Marco Notaro;,citation_author=Michael D. Evans;,citation_author=Jena S. Tronieri;,citation_author=Hannah Blau;,citation_author=Bryan Laraway;,citation_author=Tiffany J. Callahan;,citation_author=Lauren E. Chan;,citation_author=Carolyn T. Bramante;,citation_author=John B. Buse;,citation_author=Richard A. Moffitt;,citation_author=Til Stürmer;,citation_author=Steven G. Johnson;,citation_author=Yu Raymond Shao;,citation_author=Justin Reese;,citation_author=Peter N. Robinson;,citation_author=Alberto Paccanaro;,citation_author=Giorgio Valentini;,citation_author=Jared D. Huling;,citation_author=Kenneth J. Wilkins;,citation_publication_date=2023-03;,citation_cover_date=2023-03;,citation_year=2023;,citation_fulltext_html_url=https://doi.org/10.1016/j.jbi.2023.104295;,citation_doi=10.1016/j.jbi.2023.104295;,citation_volume=139;,citation_journal_title=Journal of Biomedical Informatics;,citation_publisher=Elsevier BV;">
<meta name="citation_reference" content="citation_title=Learning from data with structured missingness;,citation_author=Robin Mitra;,citation_author=Sarah F. McGough;,citation_author=Tapabrata Chakraborti;,citation_author=Chris Holmes;,citation_author=Ryan Copping;,citation_author=Niels Hagenbuch;,citation_author=Stefanie Biedermann;,citation_author=Jack Noonan;,citation_author=Brieuc Lehmann;,citation_author=Aditi Shenvi;,citation_author=Xuan Vinh Doan;,citation_author=David Leslie;,citation_author=Ginestra Bianconi;,citation_author=Ruben Sanchez-Garcia;,citation_author=Alisha Davies;,citation_author=Maxine Mackintosh;,citation_author=Eleni-Rosalina Andrinopoulou;,citation_author=Anahid Basiri;,citation_author=Chris Harbron;,citation_author=Ben D. MacArthur;,citation_publication_date=2023-01;,citation_cover_date=2023-01;,citation_year=2023;,citation_fulltext_html_url=https://doi.org/10.1038/s42256-022-00596-z;,citation_issue=1;,citation_doi=10.1038/s42256-022-00596-z;,citation_volume=5;,citation_journal_title=Nature Machine Intelligence;,citation_publisher=Springer Science; Business Media LLC;">
<meta name="citation_reference" content="citation_title=Assessment of the impact of EHR heterogeneity for clinical research through a case study of silent brain infarction;,citation_author=Sunyang Fu;,citation_author=Lester Y. Leung;,citation_author=Anne-Olivia Raulli;,citation_author=David F. Kallmes;,citation_author=Kristin A. Kinsman;,citation_author=Kristoff B. Nelson;,citation_author=Michael S. Clark;,citation_author=Patrick H. Luetmer;,citation_author=Paul R. Kingsbury;,citation_author=David M. Kent;,citation_author=Hongfang Liu;,citation_publication_date=2020-03;,citation_cover_date=2020-03;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1186/s12911-020-1072-9;,citation_issue=1;,citation_doi=10.1186/s12911-020-1072-9;,citation_volume=20;,citation_journal_title=BMC Medical Informatics and Decision Making;,citation_publisher=Springer Science; Business Media LLC;">
<meta name="citation_reference" content="citation_title=Millions of black people affected by racial bias in health-care algorithms;,citation_author=Heidi Ledford;,citation_publication_date=2019-10;,citation_cover_date=2019-10;,citation_year=2019;,citation_fulltext_html_url=https://doi.org/10.1038/d41586-019-03228-6;,citation_issue=7780;,citation_doi=10.1038/d41586-019-03228-6;,citation_volume=574;,citation_journal_title=Nature;,citation_publisher=Springer Science; Business Media LLC;">
<meta name="citation_reference" content="citation_title=Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans;,citation_author=Michael Roberts;,citation_author=Derek Driggs;,citation_author=Matthew Thorpe;,citation_author=Julian Gilbey;,citation_author=Michael Yeung;,citation_author=Stephan Ursprung;,citation_author=Angelica I. Aviles-Rivero;,citation_author=Christian Etmann;,citation_author=Cathal McCague;,citation_author=Lucian Beer;,citation_author=Jonathan R. Weir-McCall;,citation_author=Zhongzhao Teng;,citation_author=Effrossyni Gkrania-Klotsas;,citation_author=Alessandro Ruggiero;,citation_author=Anna Korhonen;,citation_author=Emily Jefferson;,citation_author=Emmanuel Ako;,citation_author=Georg Langs;,citation_author=Ghassem Gozaliasl;,citation_author=Guang Yang;,citation_author=Helmut Prosch;,citation_author=Jacobus Preller;,citation_author=Jan Stanczuk;,citation_author=Jing Tang;,citation_author=Johannes Hofmanninger;,citation_author=Judith Babar;,citation_author=Lorena Escudero Sánchez;,citation_author=Muhunthan Thillai;,citation_author=Paula Martin Gonzalez;,citation_author=Philip Teare;,citation_author=Xiaoxiang Zhu;,citation_author=Mishal Patel;,citation_author=Conor Cafolla;,citation_author=Hojjat Azadbakht;,citation_author=Joseph Jacob;,citation_author=Josh Lowe;,citation_author=Kang Zhang;,citation_author=Kyle Bradley;,citation_author=Marcel Wassin;,citation_author=Markus Holzer;,citation_author=Kangyu Ji;,citation_author=Maria Delgado Ortet;,citation_author=Tao Ai;,citation_author=Nicholas Walton;,citation_author=Pietro Lio;,citation_author=Samuel Stranks;,citation_author=Tolou Shadbahr;,citation_author=Weizhe Lin;,citation_author=Yunfei Zha;,citation_author=Zhangming Niu;,citation_author=James H. F. Rudd;,citation_author=Evis Sala;,citation_author=Carola-Bibiane Schönlieb and;,citation_publication_date=2021-03;,citation_cover_date=2021-03;,citation_year=2021;,citation_fulltext_html_url=https://doi.org/10.1038/s42256-021-00307-0;,citation_issue=3;,citation_doi=10.1038/s42256-021-00307-0;,citation_volume=3;,citation_journal_title=Nature Machine Intelligence;,citation_publisher=Springer Science; Business Media LLC;">
<meta name="citation_reference" content="citation_title=Generalisable long COVID subtypes: Findings from the NIH N3C and RECOVER programmes;,citation_author=Justin T. Reese;,citation_author=Hannah Blau;,citation_author=Elena Casiraghi;,citation_author=Timothy Bergquist;,citation_author=Johanna J. Loomba;,citation_author=Tiffany J. Callahan;,citation_author=Bryan Laraway;,citation_author=Corneliu Antonescu;,citation_author=Ben Coleman;,citation_author=Michael Gargano;,citation_author=Kenneth J. Wilkins;,citation_author=Luca Cappelletti;,citation_author=Tommaso Fontana;,citation_author=Nariman Ammar;,citation_author=Blessy Antony;,citation_author=T. M. Murali;,citation_author=J. Harry Caufield;,citation_author=Guy Karlebach;,citation_author=Julie A. McMurry;,citation_author=Andrew Williams;,citation_author=Richard Moffitt;,citation_author=Jineta Banerjee;,citation_author=Anthony E. Solomonides;,citation_author=Hannah Davis;,citation_author=Kristin Kostka;,citation_author=Giorgio Valentini;,citation_author=David Sahner;,citation_author=Christopher G. Chute;,citation_author=Charisse Madlock-Brown;,citation_author=Melissa A. Haendel;,citation_author=Peter N. Robinson;,citation_author=Heidi Spratt;,citation_author=Shyam Visweswaran;,citation_author=Joseph Eugene Flack;,citation_author=Yun Jae Yoo;,citation_author=Davera Gabriel;,citation_author=G. Caleb Alexander;,citation_author=Hemalkumar B. Mehta;,citation_author=Feifan Liu;,citation_author=Robert T. Miller;,citation_author=Rachel Wong;,citation_author=Elaine L. Hill;,citation_author=Lorna E. Thorpe;,citation_author=Jasmin Divers;,citation_publication_date=2023-01;,citation_cover_date=2023-01;,citation_year=2023;,citation_fulltext_html_url=https://doi.org/10.1016/j.ebiom.2022.104413;,citation_doi=10.1016/j.ebiom.2022.104413;,citation_volume=87;,citation_journal_title=eBioMedicine;,citation_publisher=Elsevier BV;">
<meta name="citation_reference" content="citation_title=Explainable machine learning for early assessment of COVID-19 risk prediction in emergency departments;,citation_author=Elena Casiraghi;,citation_author=Dario Malchiodi;,citation_author=Gabriella Trucco;,citation_author=Marco Frasca;,citation_author=Luca Cappelletti;,citation_author=Tommaso Fontana;,citation_author=Alessandro Andrea Esposito;,citation_author=Emanuele Avola;,citation_author=Alessandro Jachetti;,citation_author=Justin Reese;,citation_author=Alessandro Rizzi;,citation_author=Peter N. Robinson;,citation_author=Giorgio Valentini;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1109/access.2020.3034032;,citation_doi=10.1109/access.2020.3034032;,citation_volume=8;,citation_journal_title=IEEE Access;,citation_publisher=Institute of Electrical; Electronics Engineers (IEEE);">
<meta name="citation_reference" content="citation_title=Reflections on defining a standard for computable expression of scientific knowledge: What teach us yoda can;,citation_author=B. S. Alper;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36654808;,citation_issue=1;,citation_doi=10.1002/lrh2.10312;,citation_issn=2379-6146 (Electronic) 2379-6146 (Linking);,citation_volume=7;,citation_journal_title=Learn Health Syst;">
<meta name="citation_reference" content="citation_title=Testing graphical causal models using the r package &amp;amp;amp;quot;dagitty&amp;quot;;,citation_author=A. Ankan;,citation_author=I. M. N. Wortel;,citation_author=J. Textor;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/33592130;,citation_issue=2;,citation_doi=10.1002/cpz1.45;,citation_volume=1;,citation_journal_title=Curr Protoc;">
<meta name="citation_reference" content="citation_title=Long-term use of immunosuppressive medicines and in-hospital COVID-19 outcomes: A retrospective cohort study using data from the national COVID cohort collaborative;,citation_author=K. M. Andersen;,citation_author=B. A. Bates;,citation_author=E. S. Rashidi;,citation_author=A. L. Olex;,citation_author=R. B. Mannon;,citation_author=R. C. Patel;,citation_author=J. Singh;,citation_author=J. Sun;,citation_author=P. G. Auwaerter;,citation_author=D. K. Ng;,citation_author=J. B. Segal;,citation_author=B. T. Garibaldi;,citation_author=H. B. Mehta;,citation_author=G. C. Alexander;,citation_author=Covid Cohort Collaborative Consortium National;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/34806036;,citation_issue=1;,citation_doi=10.1016/S2665-9913(21)00325-8;,citation_issn=2665-9913 (Electronic) 2665-9913 (Linking);,citation_volume=4;,citation_journal_title=Lancet Rheumatol;">
<meta name="citation_reference" content="citation_title=Higher hospitalization and mortality rates among SARS-CoV-2-infected persons in rural america;,citation_author=Alfred Jerrod Anzalone;,citation_author=Ronald Horswell;,citation_author=Brian M Hendricks;,citation_author=San Chu;,citation_author=William B Hillegass;,citation_author=William H Beasley;,citation_author=Jeremy R Harper;,citation_author=Wesley Kimble;,citation_author=Clifford J Rosen;,citation_author=Lucio Miele;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_doi=10.1111/jrh.12689;,citation_volume=39;,citation_journal_title=The Journal of Rural Health;,citation_publisher=Wiley Online Library;">
<meta name="citation_reference" content="citation_title=The REporting of studies conducted using observational routinely-collected health data (RECORD) statement;,citation_author=E. I. Benchimol;,citation_author=L. Smeeth;,citation_author=A. Guttmann;,citation_author=K. Harron;,citation_author=D. Moher;,citation_author=I. Petersen;,citation_author=H. T. Sorensen;,citation_author=E. Elm;,citation_author=S. M. Langan;,citation_author=Record Working Committee;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/26440803;,citation_issue=10;,citation_doi=10.1371/journal.pmed.1001885;,citation_issn=1549-1676 (Electronic) 1549-1277 (Print) 1549-1277 (Linking);,citation_volume=12;,citation_journal_title=PLoS Med;">
<meta name="citation_reference" content="citation_title=Harmonizing units and values of quantitative data elements in a very large nationally pooled electronic health record (EHR) dataset;,citation_author=Katie R Bradwell;,citation_author=Jacob T Wooldridge;,citation_author=Benjamin Amor;,citation_author=Tellen D Bennett;,citation_author=Adit Anand;,citation_author=Carolyn Bremer;,citation_author=Yun Jae Yoo;,citation_author=Zhenglong Qian;,citation_author=Steven G Johnson;,citation_author=Emily R Pfaff;,citation_author=others;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=7;,citation_doi=10.1093/jamia/ocac054;,citation_volume=29;,citation_journal_title=Journal of the American Medical Informatics Association;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Fairness in machine learning: A survey.;,citation_author=S Caton;,citation_author=S Haas;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://arxiv.org/abs/2010.04053;,citation_doi=10.48550/arXiv.2010.0405;,citation_journal_title=arXiv;">
<meta name="citation_reference" content="citation_title=A new method of classifying prognostic comorbidity in longitudinal studies: Development and validation;,citation_author=Mary E. Charlson;,citation_author=Peter Pompei;,citation_author=Kathy L. Ales;,citation_author=C.Ronald MacKenzie;,citation_publication_date=1987;,citation_cover_date=1987;,citation_year=1987;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0021968187901718;,citation_issue=5;,citation_doi=10.1016/0021-9681(87)90171-8;,citation_issn=0021-9681;,citation_volume=40;,citation_journal_title=Journal of Chronic Diseases;">
<meta name="citation_reference" content="citation_title=Informatics playbook;,citation_author=Chunlei Wu;,citation_author=undefined C2DH;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_publisher=https://playbook.cd2h.org/;">
<meta name="citation_reference" content="citation_title=The geographies of community disaster resilience;,citation_author=SL Cutter;,citation_author=KD Ash;,citation_author=CT. Emrich;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_fulltext_html_url=https://doi.org/10.1016/j.gloenvcha.2014.08.005;,citation_issue=Nov 1;,citation_volume=29;,citation_journal_title=Global environmental change;">
<meta name="citation_reference" content="citation_title=COVID-19 TestNorm: A tool to normalize COVID-19 testing names to LOINC codes;,citation_author=Xiao Dong;,citation_author=Jianfu Li;,citation_author=Ekin Soysal;,citation_author=Jiang Bian;,citation_author=Scott L DuVall;,citation_author=Elizabeth Hanchrow;,citation_author=Hongfang Liu;,citation_author=Kristine E Lynch;,citation_author=Michael Matheny;,citation_author=Karthik Natarajan;,citation_author=others;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=9;,citation_doi=10.1093/jamia/ocaa145;,citation_volume=27;,citation_journal_title=Journal of the American Medical Informatics Association;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Real-world evidence for assessing pharmaceutical treatments in the context of COVID-19;,citation_author=J. M. Franklin;,citation_author=K. J. Lin;,citation_author=N. M. Gatto;,citation_author=J. A. Rassen;,citation_author=R. J. Glynn;,citation_author=S. Schneeweiss;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/33529354;,citation_issue=4;,citation_doi=10.1002/cpt.2185;,citation_volume=109;,citation_journal_title=Clin Pharmacol Ther;">
<meta name="citation_reference" content="citation_title=When can nonrandomized studies support valid inference regarding effectiveness or safety of new medical treatments?;,citation_author=J. M. Franklin;,citation_author=R. Platt;,citation_author=N. A. Dreyer;,citation_author=A. J. London;,citation_author=G. E. Simon;,citation_author=J. H. Watanabe;,citation_author=M. Horberg;,citation_author=A. Hernandez;,citation_author=R. M. Califf;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/33826756;,citation_issue=1;,citation_doi=10.1002/cpt.2255;,citation_issn=1532-6535 (Electronic) 0009-9236 (Print) 0009-9236 (Linking);,citation_volume=111;,citation_journal_title=Clin Pharmacol Ther;">
<meta name="citation_reference" content="citation_title=Regression and other stories;,citation_author=undefined Andrew Gelman;,citation_author=Aki Vehtari;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=http://www.cambridge.org/9781107676510;,citation_isbn=978-1107676510;">
<meta name="citation_reference" content="citation_title=Clinical concept value sets and interoperability in health data analytics;,citation_author=Sigfried Gold;,citation_author=Andrea Batch;,citation_author=Robert McClure;,citation_author=Guoqian Jiang;,citation_author=Hadi Kharrazi;,citation_author=Rishi Saripalle;,citation_author=Vojtech Huser;,citation_author=Chunhua Weng;,citation_author=Nancy Roderer;,citation_author=Ana Szarfman;,citation_author=others;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6371254;,citation_volume=2018;,citation_conference_title=AMIA annual symposium proceedings;,citation_conference=American Medical Informatics Association;">
<meta name="citation_reference" content="citation_title=Practices, norms, and aspirations regarding the construction, validation, and reuse of code sets in the analysis of real-world data;,citation_author=Sigfried Gold;,citation_author=Harold Lehmann;,citation_author=Lisa Schilling;,citation_author=Wayne Lutters;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_doi=10.1101/2021.10.14.21264917;,citation_journal_title=medRxiv;,citation_publisher=Cold Spring Harbor Laboratory Press;">
<meta name="citation_reference" content="citation_title=Collider bias undermines our understanding of COVID-19 disease risk and severity;,citation_author=G. J. Griffith;,citation_author=T. T. Morris;,citation_author=M. J. Tudball;,citation_author=A. Herbert;,citation_author=G. Mancano;,citation_author=L. Pike;,citation_author=G. C. Sharp;,citation_author=J. Sterne;,citation_author=T. M. Palmer;,citation_author=G. Davey Smith;,citation_author=K. Tilling;,citation_author=L. Zuccolo;,citation_author=N. M. Davies;,citation_author=G. Hemani;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_doi=10.1038/s41467-020-19478-2;,citation_volume=11;,citation_journal_title=Nature communications;,citation_publisher=Nature Publishing Group UK London;">
<meta name="citation_reference" content="citation_title=The National COVID Cohort Collaborative (N3C): Rationale, design, infrastructure, and deployment;,citation_abstract=Coronavirus disease 2019 (COVID-19) poses societal challenges that require expeditious data and knowledge sharing. Though organizational clinical data are abundant, these are largely inaccessible to outside researchers. Statistical, machine learning, and causal analyses are most successful with large-scale data beyond what is available in any given organization. Here, we introduce the National COVID Cohort Collaborative (N3C), an open science community focused on analyzing patient-level data from many centers.The Clinical and Translational Science Award Program and scientific community created N3C to overcome technical, regulatory, policy, and governance barriers to sharing and harmonizing individual-level clinical data. We developed solutions to extract, aggregate, and harmonize data across organizations and data models, and created a secure data enclave to enable efficient, transparent, and reproducible collaborative analytics.Organized in inclusive workstreams, we created legal agreements and governance for organizations and researchers; data extraction scripts to identify and ingest positive, negative, and possible COVID-19 cases; a data quality assurance and harmonization pipeline to create a single harmonized dataset; population of the secure data enclave with data, machine learning, and statistical analytics tools; dissemination mechanisms; and a synthetic data pilot to democratize data access.The N3C has demonstrated that a multisite collaborative learning health network can overcome barriers to rapidly build a scalable infrastructure incorporating multiorganizational clinical data for COVID-19 analytics. We expect this effort to save lives by enabling rapid collaboration among clinicians, researchers, and data scientists to identify treatments and specialized care and thereby reduce the immediate and long-term impacts of COVID-19.;,citation_author=Melissa A Haendel;,citation_author=Christopher G Chute;,citation_author=Tellen D Bennett;,citation_author=David A Eichmann;,citation_author=Justin Guinney;,citation_author=Warren A Kibbe;,citation_author=Philip R O Payne;,citation_author=Emily R Pfaff;,citation_author=Peter N Robinson;,citation_author=Joel H Saltz;,citation_author=Heidi Spratt;,citation_author=Christine Suver;,citation_author=John Wilbanks;,citation_author=Adam B Wilcox;,citation_author=Andrew E Williams;,citation_author=Chunlei Wu;,citation_author=Clair Blacketer;,citation_author=Robert L Bradford;,citation_author=James J Cimino;,citation_author=Marshall Clark;,citation_author=Evan W Colmenares;,citation_author=Patricia A Francis;,citation_author=Davera Gabriel;,citation_author=Alexis Graves;,citation_author=Raju Hemadri;,citation_author=Stephanie S Hong;,citation_author=George Hripscak;,citation_author=Dazhi Jiao;,citation_author=Jeffrey G Klann;,citation_author=Kristin Kostka;,citation_author=Adam M Lee;,citation_author=Harold P Lehmann;,citation_author=Lora Lingrey;,citation_author=Robert T Miller;,citation_author=Michele Morris;,citation_author=Shawn N Murphy;,citation_author=Karthik Natarajan;,citation_author=Matvey B Palchuk;,citation_author=Usman Sheikh;,citation_author=Harold Solbrig;,citation_author=Shyam Visweswaran;,citation_author=Anita Walden;,citation_author=Kellie M Walters;,citation_author=Griffin M Weber;,citation_author=Xiaohan Tanner Zhang;,citation_author=Richard L Zhu;,citation_author=Benjamin Amor;,citation_author=Andrew T Girvin;,citation_author=Amin Manna;,citation_author=Nabeel Qureshi;,citation_author=Michael G Kurilla;,citation_author=Sam G Michael;,citation_author=Lili M Portilla;,citation_author=Joni L Rutter;,citation_author=Christopher P Austin;,citation_author=Ken R Gersing;,citation_author=undefined N3C Consortium;,citation_publication_date=2020-08;,citation_cover_date=2020-08;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1093/jamia/ocaa196;,citation_issue=3;,citation_doi=10.1093/jamia/ocaa196;,citation_issn=1527-974X;,citation_volume=28;,citation_journal_title=Journal of the American Medical Informatics Association;">
<meta name="citation_reference" content="citation_title=Using big data to emulate a target trial when a randomized trial is not available;,citation_author=M. A. Hernan;,citation_author=J. M. Robins;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/26994063;,citation_issue=8;,citation_doi=10.1093/aje/kwv254;,citation_issn=1476-6256 (Electronic) 0002-9262 (Print) 0002-9262 (Linking);,citation_volume=183;,citation_journal_title=Am J Epidemiol;">
<meta name="citation_reference" content="citation_title=Racial disparities in COVID-19 test positivity among people living with HIV in the united states;,citation_author=J. Y. Islam;,citation_author=V. Madhira;,citation_author=J. Sun;,citation_author=A. Olex;,citation_author=N. Franceschini;,citation_author=G. Kirk;,citation_author=R. Patel;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/35306931;,citation_issue=5;,citation_doi=10.1177/09564624221074468;,citation_issn=1758-1052 (Electronic) 0956-4624 (Print) 0956-4624 (Linking);,citation_volume=33;,citation_journal_title=Int J STD AIDS;">
<meta name="citation_reference" content="citation_title=Comparing population-based risk-stratification model performance using demographic, diagnosis and medication data extracted from outpatient electronic health records versus administrative claims;,citation_author=Hadi Kharrazi;,citation_author=Winnie Chi;,citation_author=Hsien-Yen Chang;,citation_author=Thomas M Richards;,citation_author=Jason M Gallagher;,citation_author=Susan M Knudson;,citation_author=Jonathan P Weiner;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_issue=8;,citation_doi=10.1097/MLR.0000000000000754;,citation_volume=55;,citation_journal_title=Medical care;,citation_publisher=JSTOR;">
<meta name="citation_reference" content="citation_title=Crossing boundaries knowledge, disciplinarities, and interdisciplinarities;,citation_author=Julie Thompson Klein;,citation_publication_date=1996;,citation_cover_date=1996;,citation_year=1996;,citation_fulltext_html_url=https://www.google.com/books/edition/Crossing_Boundaries/bNJvYf3ROPAC;,citation_isbn=978-0813916798;,citation_series_title=Knowledge : Disciplinarity and beyond;">
<meta name="citation_reference" content="citation_title=Inherent trade-offs in the fair determination of risk scores;,citation_author=Jon M. Kleinberg;,citation_author=Sendhil Mullainathan;,citation_author=Manish Raghavan;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=http://arxiv.org/abs/1609.05807;,citation_doi=10.48550/arXiv.1609.05807;,citation_volume=abs/1609.05807;,citation_journal_title=CoRR;">
<meta name="citation_reference" content="citation_title=Literate programming;,citation_author=Donald E. Knuth;,citation_publication_date=1984-05;,citation_cover_date=1984-05;,citation_year=1984;,citation_fulltext_html_url=https://doi.org/10.1093/comjnl/27.2.97;,citation_issue=2;,citation_doi=10.1093/comjnl/27.2.97;,citation_issn=0010-4620;,citation_volume=27;,citation_journal_title=Comput. J.;,citation_publisher=Oxford University Press, Inc.;">
<meta name="citation_reference" content="citation_title=Guidance for a causal comparative effectiveness analysis emulating a target trial based on big real world evidence: When to start statin treatment;,citation_author=F. Kuehne;,citation_author=B. Jahn;,citation_author=A. Conrads-Frank;,citation_author=M. Bundo;,citation_author=M. Arvandi;,citation_author=F. Endel;,citation_author=N. Popper;,citation_author=G. Endel;,citation_author=C. Urach;,citation_author=M. Gyimesi;,citation_author=E. J. Murray;,citation_author=G. Danaei;,citation_author=T. A. Gaziano;,citation_author=A. Pandya;,citation_author=U. Siebert;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/31512926;,citation_issue=12;,citation_doi=10.2217/cer-2018-0103;,citation_issn=2042-6313 (Electronic) 2042-6305 (Linking);,citation_volume=8;,citation_journal_title=J Comp Eff Res;">
<meta name="citation_reference" content="citation_title=Use of recommended real-world methods for electronic health record data analysis has not improved over 10 years;,citation_author=Chenyu Li;,citation_author=Abdulrahman M. Alsheikh;,citation_author=Karen A. Robinson;,citation_author=Harold P. Lehmann;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.medrxiv.org/content/early/2023/06/22/2023.06.21.23291706;,citation_doi=10.1101/2023.06.21.23291706;,citation_journal_title=medRxiv;,citation_publisher=Cold Spring Harbor Laboratory Press;">
<meta name="citation_reference" content="citation_title=Clinical, social, and policy factors in COVID-19 cases and deaths: Methodological considerations for feature selection and modeling in county-level analyses;,citation_author=C. Madlock-Brown;,citation_author=K. Wilkens;,citation_author=N. Weiskopf;,citation_author=N. Cesare;,citation_author=S. Bhattacharyya;,citation_author=N. O. Riches;,citation_author=J. Espinoza;,citation_author=D. Dorr;,citation_author=K. Goetz;,citation_author=J. Phuong;,citation_author=A. Sule;,citation_author=H. Kharrazi;,citation_author=F. Liu;,citation_author=C. Lemon;,citation_author=W. G. Adams;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/35421958;,citation_issue=1;,citation_doi=10.1186/s12889-022-13168-y;,citation_issn=1471-2458 (Electronic) 1471-2458 (Linking);,citation_volume=22;,citation_journal_title=BMC Public Health;">
<meta name="citation_reference" content="citation_title=Correction: Clinical, social, and policy factors in COVID-19 cases and deaths: Methodological considerations for feature selection and modeling in county-level analyses;,citation_author=C. Madlock-Brown;,citation_author=K. Wilkens;,citation_author=N. Weiskopf;,citation_author=N. Cesare;,citation_author=S. Bhattacharyya;,citation_author=N. O. Riches;,citation_author=J. Espinoza;,citation_author=D. Dorr;,citation_author=K. Goetz;,citation_author=J. Phuong;,citation_author=A. Sule;,citation_author=H. Kharrazi;,citation_author=F. Liu;,citation_author=C. Lemon;,citation_author=W. G. Adams;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/35751109;,citation_issue=1;,citation_doi=10.1186/s12889-022-13562-6;,citation_issn=1471-2458 (Electronic) 1471-2458 (Linking);,citation_volume=22;,citation_journal_title=BMC Public Health;">
<meta name="citation_reference" content="citation_title=Use of hydroxychloroquine, remdesivir, and dexamethasone among adults hospitalized with covid-19 in the united states: A retrospective cohort study;,citation_author=Hemalkumar B. Mehta;,citation_author=Huijun An;,citation_author=Kathleen M. Andersen;,citation_author=Omar Mansour;,citation_author=Vithal Madhira;,citation_author=Emaan S. Rashidi;,citation_author=Benjamin Bates;,citation_author=Soko Setoguchi;,citation_author=Corey Joseph;,citation_author=Paul T. Kocis;,citation_author=Richard Moffitt;,citation_author=Tellen D. Bennett;,citation_author=Christopher G. Chute;,citation_author=Brian T. Garibaldi;,citation_author=G. Caleb Alexander;,citation_publication_date=2021-10-01;,citation_cover_date=2021-10-01;,citation_year=2021;,citation_issue=10;,citation_doi=10.7326/M21-0857;,citation_issn=0003-4819;,citation_volume=174;,citation_journal_title=Annals of Internal Medicine;,citation_publisher=American College of Physicians;">
<meta name="citation_reference" content="citation_title=Learning from data with structured missingness;,citation_author=Robin Mitra;,citation_author=Sarah F McGough;,citation_author=Tapabrata Chakraborti;,citation_author=Chris Holmes;,citation_author=Ryan Copping;,citation_author=Niels Hagenbuch;,citation_author=Stefanie Biedermann;,citation_author=Jack Noonan;,citation_author=Brieuc Lehmann;,citation_author=Aditi Shenvi;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_doi=10.1038/s42256-022-00596-z;,citation_volume=5;,citation_journal_title=Nature Machine Intelligence;,citation_publisher=Nature Publishing Group UK London;">
<meta name="citation_reference" content="citation_title=Identifying the PECO: A framework for formulating good questions to explore the association of environmental and other exposures with health outcomes;,citation_author=R. L. Morgan;,citation_author=P. Whaley;,citation_author=K. A. Thayer;,citation_author=H. J. Schunemann;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/30166065;,citation_issue=Pt 1;,citation_doi=10.1016/j.envint.2018.07.015;,citation_issn=1873-6750 (Electronic) 0160-4120 (Print) 0160-4120 (Linking);,citation_volume=121;,citation_journal_title=Environ Int;">
<meta name="citation_reference" content="citation_title=Increased stroke severity and mortality in patients with SARS-CoV-2 infection: An analysis from the N3C database;,citation_author=J. A. Narrett;,citation_author=I. Mallawaarachchi;,citation_author=C. M. Aldridge;,citation_author=E. D. Assefa;,citation_author=A. Patel;,citation_author=J. J. Loomba;,citation_author=S. Ratcliffe;,citation_author=O. Sadan;,citation_author=T. Monteith;,citation_author=B. B. Worrall;,citation_author=D. E. Brown;,citation_author=K. C. Johnston;,citation_author=A. M. Southerland;,citation_author=N. C. consortium;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36641948;,citation_issue=3;,citation_doi=10.1016/j.jstrokecerebrovasdis.2023.106987;,citation_issn=1532-8511 (Electronic) 1052-3057 (Print) 1052-3057 (Linking);,citation_volume=32;,citation_journal_title=J Stroke Cerebrovasc Dis;">
<meta name="citation_reference" content="citation_title=The book of OHDSI: Observational health data sciences and informatics;,citation_author=undefined OHDSI;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://ohdsi.github.io/TheBookOfOhdsi/;,citation_isbn=9781088855195;">
<meta name="citation_reference" content="citation_title=Documentation: Code repositories overview;,citation_author=undefined Palantir;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_publisher=https://www.palantir.com/docs/foundry/code-repositories/overview/;">
<meta name="citation_reference" content="citation_title=Identifying who has long COVID in the USA: A machine learning approach using N3C data.;,citation_author=E. R. Pfaff;,citation_author=A. T. Girvin;,citation_author=T. D. Bennett;,citation_author=A. Bhatia;,citation_author=I. M. Brooks;,citation_author=R. R. Deer;,citation_author=J. P. Dekermanjian;,citation_author=S. E. Jolley;,citation_author=M. G. Kahn;,citation_author=K. Kostka;,citation_author=J. A. McMurry;,citation_author=R. Moffitt;,citation_author=A. Walden;,citation_author=C. G. Chute;,citation_author=M. A. Haendel;,citation_author=C. Bramante;,citation_author=D. Dorr;,citation_author=M. Morris;,citation_author=A. M. Parker;,citation_author=H. Sidky;,citation_author=K. Gersing;,citation_author=S. Hong;,citation_author=E. Niehaus;,citation_publication_date=2022-05;,citation_cover_date=2022-05;,citation_year=2022;,citation_fulltext_html_url=https://doi.org/10.1016/S2589-7500(22)00048-6;,citation_issue=7;,citation_doi=10.1016/S2589-7500(22)00048-6;,citation_issn=0010-4620;,citation_volume=4;,citation_journal_title=Lancet Digit Health;">
<meta name="citation_reference" content="citation_title=Synergies between centralized and federated approaches to data quality: A report from the national COVID cohort collaborative;,citation_author=Emily R Pfaff;,citation_author=Andrew T Girvin;,citation_author=Davera L Gabriel;,citation_author=Kristin Kostka;,citation_author=Michele Morris;,citation_author=Matvey B Palchuk;,citation_author=Harold P Lehmann;,citation_author=Benjamin Amor;,citation_author=Mark Bissell;,citation_author=Katie R Bradwell;,citation_author=others;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_issue=4;,citation_doi=10.1093/jamia/ocab217;,citation_volume=29;,citation_journal_title=Journal of the American Medical Informatics Association;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Coding long COVID: Characterizing a new disease through an ICD-10 lens;,citation_author=Emily R Pfaff;,citation_author=Charisse Madlock-Brown;,citation_author=John M Baratta;,citation_author=Abhishek Bhatia;,citation_author=Hannah Davis;,citation_author=Andrew Girvin;,citation_author=Elaine Hill;,citation_author=Elizabeth Kelly;,citation_author=Kristin Kostka;,citation_author=Johanna Loomba;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_doi=10.1186/s12916-023-02737-6;,citation_volume=21;,citation_journal_title=BMC medicine;,citation_publisher=BioMed Central;">
<meta name="citation_reference" content="citation_title=COVID vaccine hesitancy and risk of a traffic crash;,citation_author=D. A. Redelmeier;,citation_author=J. Wang;,citation_author=D. Thiruchelvam;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36470796;,citation_issue=2;,citation_doi=10.1016/j.amjmed.2022.11.002;,citation_issn=1555-7162 (Electronic) 0002-9343 (Print) 0002-9343 (Linking);,citation_volume=136;,citation_journal_title=Am J Med;">
<meta name="citation_reference" content="citation_title=Generalisable long COVID subtypes: Findings from the NIH N3C and RECOVER programmes;,citation_author=Justin T Reese;,citation_author=Hannah Blau;,citation_author=Elena Casiraghi;,citation_author=Timothy Bergquist;,citation_author=Johanna J Loomba;,citation_author=Tiffany J Callahan;,citation_author=Bryan Laraway;,citation_author=Corneliu Antonescu;,citation_author=Ben Coleman;,citation_author=Michael Gargano;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_doi=10.1016/j.ebiom.2022.104413;,citation_volume=87;,citation_journal_title=EBioMedicine;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=Electronic health records based phenotyping in next-generation clinical trials: A perspective from the NIH health care systems collaboratory;,citation_author=Rachel L Richesson;,citation_author=W Ed Hammond;,citation_author=Meredith Nahm;,citation_author=Douglas Wixted;,citation_author=Gregory E Simon;,citation_author=Jennifer G Robinson;,citation_author=Alan E Bauck;,citation_author=Denise Cifelli;,citation_author=Michelle M Smerek;,citation_author=John Dickerson;,citation_author=others;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=e2;,citation_doi=10.1136/amiajnl-2013-001926;,citation_volume=20;,citation_journal_title=Journal of the American Medical Informatics Association;,citation_publisher=BMJ Publishing Group;">
<meta name="citation_reference" content="citation_title=Graphical depiction of longitudinal study designs in health care databases;,citation_author=S. Schneeweiss;,citation_author=J. A. Rassen;,citation_author=J. S. Brown;,citation_author=K. J. Rothman;,citation_author=L. Happe;,citation_author=P. Arlett;,citation_author=G. Dal Pan;,citation_author=W. Goettsch;,citation_author=W. Murk;,citation_author=S. V. Wang;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/30856654;,citation_issue=6;,citation_doi=10.7326/M18-3079;,citation_issn=1539-3704 (Electronic) 0003-4819 (Linking);,citation_volume=170;,citation_journal_title=Ann Intern Med;">
<meta name="citation_reference" content="citation_title=Improving reproducibility by using high-throughput observational studies with empirical calibration;,citation_author=M. J. Schuemie;,citation_author=P. B. Ryan;,citation_author=G. Hripcsak;,citation_author=D. Madigan;,citation_author=M. A. Suchard;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/30082302;,citation_issue=2128;,citation_doi=10.1098/rsta.2017.0356;,citation_issn=1471-2962 (Electronic) 1364-503X (Print) 1364-503X (Linking);,citation_volume=376;,citation_journal_title=Philos Trans A Math Phys Eng Sci;">
<meta name="citation_reference" content="citation_title=Large-scale evidence generation and evaluation across a network of databases (LEGEND): Assessing validity using hypertension as a case study;,citation_author=M. J. Schuemie;,citation_author=P. B. Ryan;,citation_author=N. Pratt;,citation_author=R. Chen;,citation_author=S. C. You;,citation_author=H. M. Krumholz;,citation_author=D. Madigan;,citation_author=G. Hripcsak;,citation_author=M. A. Suchard;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/32827027;,citation_issue=8;,citation_doi=10.1093/jamia/ocaa124;,citation_issn=1527-974X (Electronic) 1067-5027 (Print) 1067-5027 (Linking);,citation_volume=27;,citation_journal_title=J Am Med Inform Assoc;">
<meta name="citation_reference" content="citation_title=Outcomes of COVID-19 in patients with cancer: Report from the national COVID cohort collaborative (N3C);,citation_author=Noha Sharafeldin;,citation_author=Benjamin Bates;,citation_author=Qianqian Song;,citation_author=Vithal Madhira;,citation_author=Yao Yan;,citation_author=Sharlene Dong;,citation_author=Eileen Lee;,citation_author=Nathaniel Kuhrt;,citation_author=Yu Raymond Shao;,citation_author=Feifan Liu;,citation_author=Timothy Bergquist;,citation_author=Justin Guinney;,citation_author=Jing Su;,citation_author=Umit Topaloglu;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=20;,citation_doi=10.1200/JCO.21.01074;,citation_volume=39;,citation_journal_title=Journal of Clinical Oncology;">
<meta name="citation_reference" content="citation_title=Data quality considerations for evaluating COVID-19 treatments using real world data: Learnings from the national COVID cohort collaborative (N3C);,citation_author=H. Sidky;,citation_author=J. C. Young;,citation_author=A. T. Girvin;,citation_author=E. Lee;,citation_author=Y. R. Shao;,citation_author=N. Hotaling;,citation_author=S. Michael;,citation_author=K. J. Wilkins;,citation_author=S. Setoguchi;,citation_author=M. J. Funk;,citation_author=N. C. Consortium;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36800930;,citation_issue=1;,citation_doi=10.1186/s12874-023-01839-2;,citation_issn=1471-2288 (Electronic) 1471-2288 (Linking);,citation_volume=23;,citation_journal_title=BMC Med Res Methodol;">
<meta name="citation_reference" content="citation_title=Principles for data analysis workflows;,citation_author=S. Stoudt;,citation_author=V. N. Vasquez;,citation_author=C. C. Martinez;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/33735208;,citation_issue=3;,citation_doi=10.1371/journal.pcbi.1008770;,citation_issn=1553-7358 (Electronic) 1553-734X (Print) 1553-734X (Linking);,citation_volume=17;,citation_journal_title=PLoS Comput Biol;">
<meta name="citation_reference" content="citation_title=Association between immune dysfunction and COVID-19 breakthrough infection after SARS-CoV-2 vaccination in the US;,citation_author=Jing Sun;,citation_author=Qulu Zheng;,citation_author=Vithal Madhira;,citation_author=Amy L. Olex;,citation_author=Alfred J. Anzalone;,citation_author=Amanda Vinson;,citation_author=Jasvinder A. Singh;,citation_author=Evan French;,citation_author=Alison G. Abraham;,citation_author=Jomol Mathew;,citation_author=Nasia Safdar;,citation_author=Gaurav Agarwal;,citation_author=Kathryn C. Fitzgerald;,citation_author=Namrata Singh;,citation_author=Umit Topaloglu;,citation_author=Christopher G. Chute;,citation_author=Roslyn B. Mannon;,citation_author=Gregory D. Kirk;,citation_author=Rena C. Patel;,citation_publication_date=2022-02;,citation_cover_date=2022-02;,citation_year=2022;,citation_issue=2;,citation_doi=10.1001/jamainternmed.2021.7024;,citation_issn=2168-6106;,citation_volume=182;,citation_journal_title=Archives of internal medicine (Chicago, Ill. : 1908);,citation_publisher=American Medical Association;">
<meta name="citation_reference" content="citation_title=Informative missingness: What can we learn from patterns in missing laboratory data in the electronic health record?;,citation_author=A. L. M. Tan;,citation_author=E. J. Getzen;,citation_author=M. R. Hutch;,citation_author=Z. H. Strasser;,citation_author=A. Gutierrez-Sacristan;,citation_author=T. T. Le;,citation_author=A. Dagliati;,citation_author=M. Morris;,citation_author=D. A. Hanauer;,citation_author=B. Moal;,citation_author=C. L. Bonzel;,citation_author=W. Yuan;,citation_author=L. Chiudinelli;,citation_author=P. Das;,citation_author=H. G. Zhang;,citation_author=B. J. Aronow;,citation_author=P. Avillach;,citation_author=G. A. Brat;,citation_author=T. Cai;,citation_author=C. Hong;,citation_author=W. G. La Cava;,citation_author=H. Hooi Will Loh;,citation_author=Y. Luo;,citation_author=S. N. Murphy;,citation_author=K. Yuan Hgiam;,citation_author=G. S. Omenn;,citation_author=L. P. Patel;,citation_author=M. Jebathilagam Samayamuthu;,citation_author=E. R. Shriver;,citation_author=Z. Shakeri Hossein Abad;,citation_author=B. W. L. Tan;,citation_author=S. Visweswaran;,citation_author=X. Wang;,citation_author=G. M. Weber;,citation_author=Z. Xia;,citation_author=B. Verdy;,citation_author=Covid- EHR;,citation_author=Q. Long;,citation_author=D. L. Mowery;,citation_author=J. H. Holmes;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36738870;,citation_doi=10.1016/j.jbi.2023.104306;,citation_volume=139;,citation_journal_title=J Biomed Inform;">
<meta name="citation_reference" content="citation_title=Software as a medical device (SAMD): Clinical evaluation/guidance for industry and food and drug administration staff;,citation_author=U.S. Food and Drug Administration;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=https://www.fda.gov/media/100714/download;,citation_publisher=FDA;">
<meta name="citation_reference" content="citation_title=Developing real-world data and evidence to support regulatory decision-making;,citation_author=U.S. Food and Drug Administration and the Duke-Margolis Center for Health Policy;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://www.youtube.com/watch?v=-G6ltatA71I;">
<meta name="citation_reference" content="citation_title=Good machine learning practice for medical device development: Guiding principles;,citation_author=U.S. Food and Drug Administration, Health Canada, and the United Kingdom’s Medicines and Healthcare products Regulatory Agency (MHRA);,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.fda.gov/medical-devices/software-medical-device-samd/good-machine-learning-practice-medical-device-development-guiding-principles;,citation_publisher=FDA;">
<meta name="citation_reference" content="citation_title=Considerations for the design and conduct of externally controlled trials for drug and biological products guidance for industry;,citation_author=U.S. Food and Drug Administration;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.fda.gov/media/164960/download;,citation_technical_report_institution=Food; Drug Administration;">
<meta name="citation_reference" content="citation_title=The strengthening the reporting of observational studies in epidemiology (STROBE) statement: Guidelines for reporting observational studies;,citation_author=E. Elm;,citation_author=D. G. Altman;,citation_author=M. Egger;,citation_author=S. J. Pocock;,citation_author=P. C. Gotzsche;,citation_author=J. P. Vandenbroucke;,citation_author=Strobe Initiative;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/25046131;,citation_issue=12;,citation_doi=10.1016/j.ijsu.2014.07.013;,citation_issn=1743-9159 (Electronic) 1743-9159 (Linking);,citation_volume=12;,citation_journal_title=Int J Surg;">
<meta name="citation_reference" content="citation_title=Synthea™ novel coronavirus (COVID-19) model and synthetic data set;,citation_author=Jason Walonoski;,citation_author=Sybil Klaus;,citation_author=Eldesia Granger;,citation_author=Dylan Hall;,citation_author=Andrew Gregorowicz;,citation_author=George Neyarapally;,citation_author=Abigail Watson;,citation_author=Jeff Eastman;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S2666521220300077;,citation_doi=doi.org/10.1016/j.ibmed.2020.100007;,citation_issn=2666-5212;,citation_volume=1-2;,citation_journal_title=Intelligence-Based Medicine;">
<meta name="citation_reference" content="citation_title=STaRT-RWE: Structured template for planning and reporting on the implementation of real world evidence studies;,citation_author=S. V. Wang;,citation_author=S. Pinheiro;,citation_author=W. Hua;,citation_author=P. Arlett;,citation_author=Y. Uyama;,citation_author=J. A. Berlin;,citation_author=D. B. Bartels;,citation_author=K. H. Kahler;,citation_author=L. G. Bessette;,citation_author=S. Schneeweiss;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/33436424;,citation_doi=10.1136/bmj.m4856;,citation_volume=372;,citation_journal_title=BMJ;">
<meta name="citation_reference" content="citation_title=Healthcare utilization is a collider: An introduction to collider bias in EHR data reuse;,citation_author=N. G. Weiskopf;,citation_author=D. A. Dorr;,citation_author=C. Jackson;,citation_author=H. P. Lehmann;,citation_author=C. A. Thompson;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36752649;,citation_doi=10.1093/jamia/ocad013;,citation_journal_title=J Am Med Inform Assoc;">
<meta name="citation_reference" content="citation_title=R for data science;,citation_author=undefined Hadley Wickham;,citation_author=Garrett Grolemund;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://r4ds.hadley.nz/;,citation_isbn=978-1491910399;">
<meta name="citation_reference" content="citation_title=The FAIR guiding principles for scientific data management and stewardship;,citation_author=Mark D Wilkinson;,citation_author=Michel Dumontier;,citation_author=IJsbrand Jan Aalbersberg;,citation_author=Gabrielle Appleton;,citation_author=Myles Axton;,citation_author=Arie Baak;,citation_author=Niklas Blomberg;,citation_author=Jan-Willem Boiten;,citation_author=Luiz Bonino Silva Santos;,citation_author=Philip E Bourne;,citation_author=others;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_doi=10.1038/sdata.2016.18;,citation_volume=3;,citation_journal_title=Scientific data;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Knitr: A comprehensive tool for reproducible research in R;,citation_author=Yihui Xie;,citation_editor=Victoria Stodden;,citation_editor=Friedrich Leisch;,citation_editor=Roger D. Peng;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_fulltext_html_url=http://www.crcpress.com/product/isbn/9781466561595;,citation_isbn=978-1466561595;,citation_inbook_title=Implementing reproducible computational research;">
<meta name="citation_reference" content="citation_title=Dynamic documents with R and knitr;,citation_author=Yihui Xie;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=http://yihui.org/knitr/;,citation_isbn=978-1498716963;">
<meta name="citation_reference" content="citation_title=Bookdown: Authoring books and technical documents with R markdown;,citation_author=Yihui Xie;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=https://bookdown.org/yihui/bookdown;,citation_isbn=978-1138700109;">
<meta name="citation_reference" content="citation_title=R markdown: The definitive guide;,citation_author=Yihui Xie;,citation_author=J. J. Allaire;,citation_author=Garrett Grolemund;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://bookdown.org/yihui/rmarkdown;,citation_isbn=978-1138359338;">
<meta name="citation_reference" content="citation_title=R markdown cookbook;,citation_author=Yihui Xie;,citation_author=Christophe Dervieux;,citation_author=Emily Riederer;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://bookdown.org/yihui/rmarkdown-cookbook;,citation_isbn=9780367563837;">
<meta name="citation_reference" content="citation_title=Associations between HIV infection and clinical spectrum of COVID-19: A population level analysis based on US national COVID cohort collaborative (N3C) data;,citation_author=Xueying Yang;,citation_author=Jing Sun;,citation_author=Rena C Patel;,citation_author=Jiajia Zhang;,citation_author=Siyuan Guo;,citation_author=Qulu Zheng;,citation_author=Amy L Olex;,citation_author=Bankole Olatosi;,citation_author=Sharon B Weissman;,citation_author=Jessica Y Islam;,citation_author=others;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=11;,citation_doi=10.1016/S2352-3018(21)00239-3;,citation_volume=8;,citation_journal_title=The Lancet HIV;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=Comparative effectiveness of dexamethasone in treatment of hospitalized COVID-19 patients during the first year of the pandemic: The N3C data repository;,citation_author=R. Zhou;,citation_author=K. E. Johnson;,citation_author=J. F. Rousseau;,citation_author=P. J. Rathouz;,citation_author=N. C. Consortium;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/36324806;,citation_doi=10.1101/2022.10.22.22281373;,citation_journal_title=medRxiv;">
</head>

<body class="nav-sidebar floating quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../representation/cdm.html">Uniform Data Representation</a></li><li class="breadcrumb-item"><a href="../representation/vocabularies.html"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Standardized Vocabularies</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../representation/cdm.html">Uniform Data Representation</a></li><li class="breadcrumb-item"><a href="../representation/vocabularies.html"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Standardized Vocabularies</span></a></li></ol></nav>
      <h1 class="title"><span id="sec-representation-vocabularies" class="quarto-section-identifier"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Standardized Vocabularies</span></span></h1>
                      </div>
  </div>
    
  
  <div class="quarto-title-meta">

      
    
      
    </div>
    
  <div>
    <div class="abstract">
      <div class="block-title">Abstract</div>
      TODO- write abstract
    </div>
  </div>
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">The Book of OHDSI 2nd Edition</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" role="link" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="../The-Book-of-OHDSI-2nd-Edition.pdf">
              <i class="bi bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="../The-Book-of-OHDSI-2nd-Edition.epub">
              <i class="bi bi-journal pe-1"></i>
            Download ePub
            </a>
          </li>
      </ul>
    </div>
    <div class="dropdown">
      <a href="" title="Share" id="quarto-navigation-tool-dropdown-1" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" role="link" aria-label="Share"><i class="bi bi-share"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-1">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
              <i class="bi bi-twitter pe-1"></i>
            Twitter
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
              <i class="bi bi-facebook pe-1"></i>
            Facebook
            </a>
          </li>
      </ul>
    </div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">OHDSI</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../ohdsi/community.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">The OHDSI Community</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../ohdsi/principles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">OHDSI Principles</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../ohdsi/begin.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Where to Begin</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Uniform Data Representation</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../representation/cdm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">The Common Data Model</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../representation/vocabularies.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Standardized Vocabularies</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../representation/etl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Extract Transform Load</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../representation/sources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Data Sources</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true">
 <span class="menu-text">Data Analytics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/cases.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Data Analytics Use Cases</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/tools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">OHDSI Analytics Tools</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/sqlr.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">SQL and R</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/cohorts.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Defining Cohorts</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/characterization.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Characterization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/population.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Population-Level Estimation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../analytics/patient.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Patient-Level Prediction</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">Evidence Quality</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../quality/evidence.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Evidence Quality</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../quality/data.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Data Quality</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../quality/clinical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Clinical Validity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../quality/software.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Software Validity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../quality/diagnostics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Diagnostics</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">OHDSI Research</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../research/steps.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Study Steps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../research/network.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">OHDSI Network Research</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../research/engagement.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Engagement with Networks</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">OHDSI in Action</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../action/therapeutic.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Study Steps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../action/genai.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Generative AI</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true">
 <span class="menu-text">Back Matter</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/protocol.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Protocol Template</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#why-vocabularies-and-why-standardizing" id="toc-why-vocabularies-and-why-standardizing" class="nav-link active" data-scroll-target="#why-vocabularies-and-why-standardizing"><span class="header-section-number">5.1</span> Why Vocabularies, and Why Standardizing</a>
  <ul class="collapse">
  <li><a href="#vocabularies-use-cases-and-users" id="toc-vocabularies-use-cases-and-users" class="nav-link" data-scroll-target="#vocabularies-use-cases-and-users"><span class="header-section-number">5.1.1</span> Vocabularies Use Cases and Users</a></li>
  <li><a href="#access-to-the-standardized-vocabularies" id="toc-access-to-the-standardized-vocabularies" class="nav-link" data-scroll-target="#access-to-the-standardized-vocabularies"><span class="header-section-number">5.1.2</span> Access to the Standardized Vocabularies</a></li>
  </ul></li>
  <li><a href="#vocabularies-process-and-governance" id="toc-vocabularies-process-and-governance" class="nav-link" data-scroll-target="#vocabularies-process-and-governance"><span class="header-section-number">5.2</span> Vocabularies Process and Governance</a>
  <ul class="collapse">
  <li><a href="#building-the-standardized-vocabularies-and-vocabularies-principles" id="toc-building-the-standardized-vocabularies-and-vocabularies-principles" class="nav-link" data-scroll-target="#building-the-standardized-vocabularies-and-vocabularies-principles"><span class="header-section-number">5.2.1</span> Building the Standardized Vocabularies and Vocabularies Principles</a></li>
  <li><a href="#vocabularies-governance-roadmap-and-role-of-the-community" id="toc-vocabularies-governance-roadmap-and-role-of-the-community" class="nav-link" data-scroll-target="#vocabularies-governance-roadmap-and-role-of-the-community"><span class="header-section-number">5.2.2</span> Vocabularies Governance, Roadmap and Role of the Community</a></li>
  </ul></li>
  <li><a href="#ohdsi-vocabularies-structure-concepts-and-relationships" id="toc-ohdsi-vocabularies-structure-concepts-and-relationships" class="nav-link" data-scroll-target="#ohdsi-vocabularies-structure-concepts-and-relationships"><span class="header-section-number">5.3</span> OHDSI Vocabularies Structure: Concepts and Relationships</a>
  <ul class="collapse">
  <li><a href="#concept-ids" id="toc-concept-ids" class="nav-link" data-scroll-target="#concept-ids"><span class="header-section-number">5.3.1</span> Concept IDs</a></li>
  <li><a href="#concept-names" id="toc-concept-names" class="nav-link" data-scroll-target="#concept-names"><span class="header-section-number">5.3.2</span> Concept Names</a></li>
  <li><a href="#domains" id="toc-domains" class="nav-link" data-scroll-target="#domains"><span class="header-section-number">5.3.3</span> Domains</a></li>
  <li><a href="#vocabularies" id="toc-vocabularies" class="nav-link" data-scroll-target="#vocabularies"><span class="header-section-number">5.3.4</span> Vocabularies</a></li>
  <li><a href="#concept-classes" id="toc-concept-classes" class="nav-link" data-scroll-target="#concept-classes"><span class="header-section-number">5.3.5</span> Concept Classes</a></li>
  <li><a href="#standard-concepts" id="toc-standard-concepts" class="nav-link" data-scroll-target="#standard-concepts"><span class="header-section-number">5.3.6</span> Standard Concepts</a></li>
  <li><a href="#non-standard-concepts" id="toc-non-standard-concepts" class="nav-link" data-scroll-target="#non-standard-concepts"><span class="header-section-number">5.3.7</span> Non-Standard Concepts</a></li>
  <li><a href="#classification-concepts" id="toc-classification-concepts" class="nav-link" data-scroll-target="#classification-concepts"><span class="header-section-number">5.3.8</span> Classification Concepts</a></li>
  <li><a href="#concept-codes" id="toc-concept-codes" class="nav-link" data-scroll-target="#concept-codes"><span class="header-section-number">5.3.9</span> Concept Codes</a></li>
  <li><a href="#lifecycle" id="toc-lifecycle" class="nav-link" data-scroll-target="#lifecycle"><span class="header-section-number">5.3.10</span> Lifecycle</a></li>
  <li><a href="#relationships" id="toc-relationships" class="nav-link" data-scroll-target="#relationships"><span class="header-section-number">5.3.11</span> Relationships</a></li>
  <li><a href="#mapping-relationships" id="toc-mapping-relationships" class="nav-link" data-scroll-target="#mapping-relationships"><span class="header-section-number">5.3.12</span> Mapping Relationships</a></li>
  <li><a href="#hierarchical-relationships-and-hierarchy" id="toc-hierarchical-relationships-and-hierarchy" class="nav-link" data-scroll-target="#hierarchical-relationships-and-hierarchy"><span class="header-section-number">5.3.13</span> Hierarchical Relationships and Hierarchy</a></li>
  <li><a href="#other-relationships" id="toc-other-relationships" class="nav-link" data-scroll-target="#other-relationships"><span class="header-section-number">5.3.14</span> Other Relationships</a></li>
  </ul></li>
  <li><a href="#special-situations" id="toc-special-situations" class="nav-link" data-scroll-target="#special-situations"><span class="header-section-number">5.4</span> Special Situations</a>
  <ul class="collapse">
  <li><a href="#device-coding" id="toc-device-coding" class="nav-link" data-scroll-target="#device-coding"><span class="header-section-number">5.4.1</span> Device Coding</a></li>
  <li><a href="#coding-in-oncology" id="toc-coding-in-oncology" class="nav-link" data-scroll-target="#coding-in-oncology"><span class="header-section-number">5.4.2</span> Coding in Oncology</a></li>
  <li><a href="#coding-in-psychiatry" id="toc-coding-in-psychiatry" class="nav-link" data-scroll-target="#coding-in-psychiatry"><span class="header-section-number">5.4.3</span> Coding in Psychiatry</a></li>
  <li><a href="#coding-for-gis-exposomes-and-sdoh" id="toc-coding-for-gis-exposomes-and-sdoh" class="nav-link" data-scroll-target="#coding-for-gis-exposomes-and-sdoh"><span class="header-section-number">5.4.4</span> Coding for GIS, Exposomes and SDOH</a></li>
  <li><a href="#microbiology-and-susceptibility-coding" id="toc-microbiology-and-susceptibility-coding" class="nav-link" data-scroll-target="#microbiology-and-susceptibility-coding"><span class="header-section-number">5.4.5</span> Microbiology and Susceptibility Coding</a></li>
  <li><a href="#survey-coding" id="toc-survey-coding" class="nav-link" data-scroll-target="#survey-coding"><span class="header-section-number">5.4.6</span> Survey Coding</a></li>
  <li><a href="#flavors-of-null" id="toc-flavors-of-null" class="nav-link" data-scroll-target="#flavors-of-null"><span class="header-section-number">5.4.7</span> Flavors of NULL</a></li>
  </ul></li>
  <li><a href="#summary" id="toc-summary" class="nav-link" data-scroll-target="#summary"><span class="header-section-number">5.5</span> Summary</a></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/edit/main/representation/vocabularies.md" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/blob/main/representation/vocabularies.md" class="toc-action"><i class="bi empty"></i>View source</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">






<p><strong>Chapter Leads</strong>: Anna Ostropolets</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>This is page is currently a stub. The chapter is being written in the OHDSI Teams directory. When the draft is complete, it will be converted to markdown and moved to this file.</p>
<p>Author Resources (requires an OHDSI Teams account):</p>
<ul>
<li><a href="https://ohdsiorg.sharepoint.com/sites/Workgroup-EducationWorkingGroup/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FWorkgroup%2DEducationWorkingGroup%2FShared%20Documents%2FSection%20II%20%2D%20Data%2FChapter%205%20%2D%20Standardized%20Vocabularies&amp;viewid=05fec2cc%2Dec8a%2D4d04%2Db565%2Dcf1289b96f67">Chapter 5 Directory</a></li>
<li><a href="https://ohdsiorg.sharepoint.com/:x:/r/sites/Workgroup-EducationWorkingGroup/_layouts/15/Doc2.aspx?action=edit&amp;sourcedoc=%7B1fa31e39-1c5f-4918-b878-609ebd9810b3%7D&amp;wdOrigin=TEAMS-WEB.teamsSdk_ns.rwc&amp;wdExp=TEAMS-TREATMENT&amp;wdhostclicktime=1748104477731&amp;web=1">Book Layout</a></li>
<li><a href="https://ohdsiorg.sharepoint.com/sites/Workgroup-EducationWorkingGroup/Shared%20Documents/Forms/AllItems.aspx?viewid=05fec2cc%2Dec8a%2D4d04%2Db565%2Dcf1289b96f67">Education Working Group SharePoint Drive</a></li>
</ul>
<p>Public Resources:</p>
<ul>
<li><a href="https://ohdsi.github.io/TheBookOfOhdsi/">Book of OHDSI, Edition 1</a></li>
<li><a href="https://github.com/OHDSI/TheBookOfOhdsi">Source Code</a> for Book of OHDSI, Edition 1</li>
<li><a href="https://ohdsi.org/">OHDSI Home Page</a></li>
</ul>
</div>
</div>
<p>The OHDSI Standardized Vocabularies are a foundational part of the OHDSI research network, and an integral part of the Common Data Model (CDM) (1). They allow standardization of methods, definitions, and results by defining the content of the data, paving the way for true remote (behind the firewall) network research and analytics. Usually, finding and interpreting the content of observational healthcare data, whether it is structured data using coding schemes or laid down in free text, is passed all the way through to the researcher, who is faced with a myriad of different ways to describe clinical events. OHDSI requires harmonization not only to a standardized format, but also to a rigorous standard content.</p>
<p>The Vocabularies are a collection of public standard vocabularies and terminologies used in the network, which we consolidate from their different original formats and life-cycle conventions into the CDM table structure.&nbsp;The system is dynamic, it evolves with frequent source vocabulary updates, deprecations, and concept replacements, all of which are version-controlled and available via ATHENA - OHDSI’s vocabulary distribution platform (2). Vocabularies support phenotyping, covariate construction, large-scale analytics and result reporting and is a product of community effort: Vocabulary Team maintains and improves vocabularies according to the roadmap (3), vocabulary stewards maintain individual vocabularies (4), various Workgroups coordinate efforts for special use cases such as device or vaccine harmonization and contributors across the community add their vocabulary content and improve existing, particularly through enhancing mappings (5).</p>
<p>In this chapter we first describe the main principles of the Standardized Vocabularies and processes. We will walk through Vocabularies components and some typical situations, all of which are necessary to understand and utilize this foundational resource. We also point out where the support of the community is required to continuously improve it and describe special situations that require further development.</p>
<section id="why-vocabularies-and-why-standardizing" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="why-vocabularies-and-why-standardizing"><span class="header-section-number">5.1</span> Why Vocabularies, and Why Standardizing</h2>
<p>Medical vocabularies go back to the Bills of Mortality in medieval London to manage outbreaks of the plague and other diseases (<a href="#fig-representation-vocabularies-010-london" class="quarto-xref">Figure&nbsp;<span>5.1</span></a>).</p>
<div id="fig-representation-vocabularies-010-london" class="quarto-float quarto-figure quarto-figure-center anchored" alt="London Mortality Bill">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-representation-vocabularies-010-london-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/vocabularies/fig-representation-vocabularies-010-london.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;5.1: 1660 London Bill of Mortality, showing the cause of death for deceased inhabitants using a classification system of 62 diseases known at the time."><img src="images/vocabularies/fig-representation-vocabularies-010-london.jpg" class="img-fluid figure-img" alt="London Mortality Bill"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-representation-vocabularies-010-london-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.1: 1660 London Bill of Mortality, showing the cause of death for deceased inhabitants using a classification system of 62 diseases known at the time.
</figcaption>
</figure>
</div>
<p>Since then, the classifications have greatly expanded in size and complexity and spread into other aspects of healthcare, such as procedures and services, drugs, medical devices, etc. The main principles have remained the same: they are controlled vocabularies, terminologies, hierarchies, or ontologies that some healthcare communities agree upon for the purpose of capturing, classifying, and analyzing patient data. Many of these vocabularies are maintained by public and government agencies with a long-term mandate for doing so. For example, the World Health Organization (WHO) produces the International Classification of Disease (ICD) with the recent addition of its 11th revision (ICD11). Local governments create country-specific versions, such as ICD10CM (USA), ICD10GM (Germany), etc. Governments also control the marketing and sale of drugs and maintain national repositories of such certified drugs. Vocabularies are also used in the private sector, either as commercial products or for internal use, such as electronic health record (EHR) systems or for medical insurance claim reporting.</p>
<p>As a result, each country, region, healthcare system and institution tend to have their own classifications that would most likely only be relevant where it is used. This myriad of vocabularies prevents interoperability of the systems they are used in. Standardization is the key that enables patient data exchange, unlocks health data analysis on a global level, and allows systematic and standardized research, including performance characterization and quality assessment. To address the interoperability problem, multinational organizations have sprung up and started creating broad standards, such as the Standard Nomenclature of Medicine (SNOMED) and Logical Observation Identifiers Names and Codes (LOINC). In the US, the Health IT Standards Committee (HITAC) recommends the use of SNOMED, LOINC, and the drug vocabulary RxNorm as standards to the National Coordinator for Health IT (ONC) for use in a common platform for nationwide health information exchange across diverse entities.</p>
<p>OHDSI developed the OMOP CDM, a global standard for observational research. As part of the CDM, the OHDSI Standardized Vocabularies are available for two main purposes:</p>
<ul>
<li><p>Common repository of all vocabularies used in the community</p></li>
<li><p>Standardization and mapping for use in research</p></li>
</ul>
<p>The Standardized Vocabularies are available to the community free of charge and&nbsp;<strong>must be used</strong>&nbsp;for OMOP CDM instance <strong>as its mandatory reference table</strong>. It is crucial to use the most recent version of the Vocabularies and continuously incorporate new versions in the ETL as Vocabularies changes and shifts impact common research tasks (6).</p>
<section id="vocabularies-use-cases-and-users" class="level3" data-number="5.1.1">
<h3 data-number="5.1.1" class="anchored" data-anchor-id="vocabularies-use-cases-and-users"><span class="header-section-number">5.1.1</span> Vocabularies Use Cases and Users</h3>
<p>OHDSI Vocabularies are different from other ontology systems, such as the Unified Medical Language System (UMLS) (7) and the difference stems from the main use case of evidence generation that OHDSI supports. Both UMLS and OHDSI aggregate relationships from source vocabularies. UMLS provides crosswalks among vocabularies with various degrees of fidelity and such crosswalks can be incomplete or ambiguous. OHDSI curates mappings and selects high-quality ones for the official “Maps to” relationships from source vocabularies to a single reference standard, ensuring that all data sources speak the same language. For example, “Atrial fibrillation” from ICD-9, Read, MeSH, etc., are all mapped to a single SNOMED concept in the Condition domain. UMLS, in contrast, groups synonymous terms under a CUI but does not designate one as “the code to use” serving as a translation table and not enforcing a single vocabulary for data encoding. UMLS contains many international vocabularies, but historically it has had a strong U.S. focus, and some content can lag in updates. OHDSI Vocabularies explicitly integrate both US and non-US coding systems and even create new standard concepts for non-US use cases to achieve global coverage. For example, US drugs are covered in RxNorm that we import, international drugs are covered in RxNorm Extension that we create de-novo and both of them are integrated with Anatomic Therapeutic Classification (ATC) (8,9). OHDSI Vocabularies are optimized for standardized analytics, offering open-access, harmonized coverage of both U.S. and international terminologies to enable consistent, reproducible studies across institutions and countries.</p>
<p>Vocabularies are centered around generating real-world evidence from observational studies and are mostly used for two groups of tasks: ETL of data to OMOP CDM and subsequent research on converted data. If you are a data engineer/ETL developer, the most relevant information is how to use correct source-to-standard mappings and populate both standard and source concept ID fields appropriately. Additionally, you need to know how to track vocabulary changes adopt ETL accordingly. If you are a researcher, the most relevant information is how to use vocabularies to find relevant codes for concept sets and features, use hierarchies, and examine mappings.</p>
</section>
<section id="access-to-the-standardized-vocabularies" class="level3" data-number="5.1.2">
<h3 data-number="5.1.2" class="anchored" data-anchor-id="access-to-the-standardized-vocabularies"><span class="header-section-number">5.1.2</span> Access to the Standardized Vocabularies</h3>
<p>The OHDSI Standardized Vocabularies are distributed via ATHENA (2), a web-based platform for browsing and downloading vocabulary data. You can use it to search, explore, and filter vocabularies by domain, concept class, vocabulary source, standard status and validity. You can select relevant vocabularies and download a pre-packaged vocabulary bundle, ready for loading into a local OMOP CDM instance.</p>
<p>To download a zip file with all Standardized Vocabularies tables, select all the vocabularies you need for your OMOP CDM. Vocabularies with Standard Concepts and very common usage are preselected. Add vocabularies that are used in your source data. Vocabularies that are proprietary have no select button. Click on the “License required” button to incorporate a licensed-required vocabulary into your list. The Vocabulary Team will contact you and request you demonstrate your license or help you connect to the right body to obtain one.</p>
<p>Each vocabulary download includes a ZIP file containing a standard set of CSV files, which can be loaded into your database using standard SQL scripts or programmatically. You will also need to re-constitute names of CPT4 codes as per our use agreement (10).</p>
<p>The VOCABULARY.csv file contains the version and release date metadata for each vocabulary, which should be recorded to ensure reproducibility in analyses and network studies. When updating to a newer vocabulary version, we recommend reviewing the changes in concept definitions, domain assignments, mappings, and deprecated concepts to ensure that downstream data and cohort definitions remain valid (6).</p>
<p>You can also select a specific vocabulary release different from the current release or download a file that contains the delta between two given releases.</p>
</section>
</section>
<section id="vocabularies-process-and-governance" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="vocabularies-process-and-governance"><span class="header-section-number">5.2</span> Vocabularies Process and Governance</h2>
<section id="building-the-standardized-vocabularies-and-vocabularies-principles" class="level3" data-number="5.2.1">
<h3 data-number="5.2.1" class="anchored" data-anchor-id="building-the-standardized-vocabularies-and-vocabularies-principles"><span class="header-section-number">5.2.1</span> Building the Standardized Vocabularies and Vocabularies Principles</h3>
<p>All vocabularies of the Standardized Vocabularies are consolidated into the same common format: <code>CONCEPT</code>, <code>CONCEPT_RELATIONSHIP</code>, <code>CONCEPT_ANCESTOR</code>, <code>CONCEPT_SYNONYM</code>, and supporting reference files such as VOCABULARY, DOMAIN, CONCEPT_CLASS, and RELATIONSHIP. This relieves the researchers from having to understand and handle multiple different formats and life-cycle conventions of the originating vocabularies. <mark></mark></p>
<p>OHDSI generally prefers adopting existing vocabularies, rather than de-novo construction, because (i) many vocabularies have already been utilized in observational data in the community, and (ii) construction and maintenance of vocabularies is complex and requires the input of many stakeholders over long periods of time to mature. For this reason, dedicated organizations provide vocabularies, which undergo a life cycle of generation, deprecation, merging, and splitting. Currently, OHDSI only produces internal administrative vocabularies like Type Concepts (for example,&nbsp;condition type concepts) as well as several other vocabularies to cover areas with existing gaps: RxNorm Extension to cover drugs that are only used outside the United States, OMOP Investigational Drugs for investigational drugs, Cancer Modifier for cancer measurements, and OMOP Extension for miscellaneous gaps. There are other community-driven efforts, such as GIS Vocabulary Package (11).</p>
<p>All vocabularies go through several common stages upon refresh: staging or harmonization to a common table structure, normalization and creation of crosswalks, integration with other vocabularies and release (12). All steps are accompanied by a set of quality assurance and control procedures, both automated and human-curated (13).</p>
<p>OHDSI Vocabularies follow twelve principles (14). Among others, Vocabularies focus on and support OHDSI <strong>use case</strong> of generating new evidence. They meant to be&nbsp;<strong>comprehensive,</strong> that is&nbsp;there are enough concepts to cover any event relevant to the patient’s healthcare experience (e.g.,&nbsp;conditions, procedures, exposures to drug, etc.) as well as some of the administrative information of the healthcare system (e.g.,&nbsp;visits, care sites, etc.). They strive to have <strong>unique standard concept</strong>, where for each Clinical Entity there is only one concept representing it, called the Standard Concept. Other equivalent or similar concepts are designated non-Standard and mapped to the Standard ones. Moreover, such concepts should be stated as fact, no negations of facts, no reference to the past, and no flavors of null (unknown, not reported, etc.).</p>
</section>
<section id="vocabularies-governance-roadmap-and-role-of-the-community" class="level3" data-number="5.2.2">
<h3 data-number="5.2.2" class="anchored" data-anchor-id="vocabularies-governance-roadmap-and-role-of-the-community"><span class="header-section-number">5.2.2</span> Vocabularies Governance, Roadmap and Role of the Community</h3>
<p>OHDSI Vocabularies work and processes are governed by the OHDSI Central Coordinating Center’s body, Vocabulary Committee, which includes representatives from across the OHDSI community and helps set priorities for maintenance, content expansion, and quality improvement. Committee’s work is based on the landscape assessment conducted in 2023 (15). As a part of this work, it approves the roadmap for bi-annual releases of the Vocabularies.</p>
<p>Release happens in February and August and is accompanied by detailed roadmap updates and release notes describing the changes (16). Each release note describes (1) newly added vocabularies, (2) concepts and/or mappings newly added to the existing vocabularies, (4) changes in mappings, domains, status of the concepts as well as detailed description of the actions performed for specific vocabularies. Additionally, the release notes contain the artifacts not available through Athena: pack content, SSSOM-compatible metadata (17) for concepts and relationships as well as reports for alignment with vocabulary principles (August 2024 release).</p>
<p>You can use information about releases and the roadmap in two ways. First, you can assess the content of each release and use open source tools to assess its impact on ETL and research (6,18). If you are a ETLer or are responsible for your institution’s data/OMOP CDM, <strong>you should update OMOP CDM instance with the latest version of the Vocabularies</strong> to benefit from improved coverage, consistency, and corrected mappings. Vocabularies <strong>change</strong> a lot. If you are not updating, you are falling behind in research.</p>
<p>Second, you can use this information to assess if planned activities meet your needs. You assist in vocabulary maintenance if your vocabulary is not on the roadmap as a vocabulary steward (19). With each release stewards from the community refresh and improve their vocabularies, even if they are not on the roadmap (current list of stewards can be found here (4)). You can also add your vocabulary, concepts or improve existing content (mappings, domains) through community contribution (5).</p>
<section id="community-contributions" class="level4" data-number="5.2.2.1">
<h4 data-number="5.2.2.1" class="anchored" data-anchor-id="community-contributions"><span class="header-section-number">5.2.2.1</span> Community contributions</h4>
<p>The extensive scope of the OMOP Standard Vocabularies poses a challenge to maintenance and scalability. The OHDSI Vocabulary Team focuses on core terminologies out of necessity. However, there are plenty of opportunities for the OHDSI community to assist in vocabulary maintenance. Examples above regarding improvements in mappings, labels, synonyms, etc. are welcomed from the community. Working Groups may feel particular ownership of a domain or specialty area and wish to help manage the necessary vocabulary. When this happens, the core terminology management system can be extended through integration of community-contributed content. Begun in earnest in 2024, this community and centrally-management vocabulary integration allows for more scalable contribution and more rapid conceptual gap-filling. A community-contribution infrastructure has been developed in phases depending on the complexity of the contribution. Small changes or additions can be provided using templates.</p>
<p>You can use templates to add a new standard vocabulary, add non-standard concepts, add mappings, change mappings or concept domains, or propose upgrading a non-standard concept to standard. Modification of content requires community approval through the Vocabulary Workgroup. Template submissions should be completed and ratified two months prior to the release date (end of June/end of December for August and February releases, respectively). Instructions for completed templates are described on the GitHub Wiki (5).</p>
<p>Larger contributions (for example, entire terminologies or drug catalogues) require staging and integration using a compatible environment to that used for managing the core terminologies. Examples of community-staged contributions include the Heme-Onc vocabulary, the Veterinary vocabulary and the CIEL terminology. For complex contributions, it is best to have a working group sponsor your request. You can use the instructions provided on Wiki under Community Contributions Part II (5). We recommend you talk to the members of the Vocabulary Workgroup or Team to discuss your specific use case.</p>
</section>
</section>
</section>
<section id="ohdsi-vocabularies-structure-concepts-and-relationships" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="ohdsi-vocabularies-structure-concepts-and-relationships"><span class="header-section-number">5.3</span> OHDSI Vocabularies Structure: Concepts and Relationships</h2>
<p>All clinical events in the OMOP CDM are represented as concepts, which capture the semantic notion of each event. They are the fundamental building blocks of the data records, making almost all tables fully normalized with few exceptions. Concepts are stored in the <code>CONCEPT</code> table (<a href="#fig-representation-vocabularies-020-standard-mapping" class="quarto-xref">Figure&nbsp;<span>5.2</span></a>).</p>
<div id="fig-representation-vocabularies-020-standard-mapping" class="quarto-float quarto-figure quarto-figure-center anchored" alt="standard-mapping">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-representation-vocabularies-020-standard-mapping-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/vocabularies/fig-representation-vocabularies-020-standard-mapping.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;5.2: Standard representation of vocabulary concepts in the OMOP CDM. The example provided is the CONCEPT table record for the SNOMED code for Atrial Fibrillation."><img src="images/vocabularies/fig-representation-vocabularies-020-standard-mapping.png" class="img-fluid figure-img" alt="standard-mapping"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-representation-vocabularies-020-standard-mapping-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.2: Standard representation of vocabulary concepts in the OMOP CDM. The example provided is the <code>CONCEPT</code> table record for the SNOMED code for Atrial Fibrillation.
</figcaption>
</figure>
</div>
<section id="concept-ids" class="level3" data-number="5.3.1">
<h3 data-number="5.3.1" class="anchored" data-anchor-id="concept-ids"><span class="header-section-number">5.3.1</span> Concept IDs</h3>
<p>Each concept is assigned a concept ID to be used as a primary key. This meaningless integer ID, rather than the original code from the source vocabulary, is used to record data in the CDM event tables via the foreign key fields. No two concepts (even from different vocabularies) share the same ID. Conversely, the same source code might appear in multiple vocabularies, but each distinct concept gets its own ID.</p>
</section>
<section id="concept-names" class="level3" data-number="5.3.2">
<h3 data-number="5.3.2" class="anchored" data-anchor-id="concept-names"><span class="header-section-number">5.3.2</span> Concept Names</h3>
<p>Each concept has one name. Names are always in English. They are imported from the source of the vocabulary. If the source vocabulary has more than one name, the most expressive (fully specified) is selected and the remaining ones are stored in the <code>CONCEPT_SYNONYM</code> table under the same <code>CONCEPT_ID</code> key. Non-English names are recorded in <code>CONCEPT_SYNONYM</code> as well, with the appropriate language concept ID in the <code>LANGUAGE_CONCEPT_ID</code> field. The name can only be 255 characters long, which means that very long names get truncated, and the full-length version recorded as another synonym, which can hold up to 1000 characters. Tools like Athena and ATLAS use the concept names and synonyms to let users search for concepts. When doing analysis, it is often convenient to have the concept names for interpretability, but analysis logic should use the <code>CONCEPT_ID</code>.</p>
</section>
<section id="domains" class="level3" data-number="5.3.3">
<h3 data-number="5.3.3" class="anchored" data-anchor-id="domains"><span class="header-section-number">5.3.3</span> Domains</h3>
<p>Each concept is assigned a domain in the <code>DOMAIN_ID</code> field, which, in contrast to the numerical <code>CONCEPT_ID</code>, is a short, case-sensitive, unique alphanumeric ID for the domain. Domains are OMOP-specific and correspond to the OMOP CDM tables (20). Examples of such identifiers are “Condition,” “Drug,” “Procedure,” “Visit,” “Device,” “Specimen,” etc. Domains also direct to which CDM table and field a clinical event or event attribute is recorded. For example, “Atrial fibrillation” is a clinical finding that would be recorded in the Condition Occurrence table, so its domain is “Condition”; a concept for a lab test (for example, “Blood glucose measurement”) would have domain “Measurement” and belong in the Measurement table. Domains are assigned to codes, and a vocabulary can have different domains: for example, HCPCS, while considered procedure vocabulary, also has codes with Drug and Observation domains.</p>
<p>The domain heuristic follows the definitions of the domains. These definitions are derived from the table and field definitions in the CDM <strong>{Chapter 4}</strong>. The heuristic is not perfect; there are grey zones (<strong>{Section 5.4}&nbsp;</strong>”Special Situations”), source vocabulary shifts, and occasional misassignments. Although domains of concepts may change, 95% of the concepts never changed their domain since Vocabularies’ inception (for more information, see Assets in v20240830 release notes) (16).</p>
</section>
<section id="vocabularies" class="level3" data-number="5.3.4">
<h3 data-number="5.3.4" class="anchored" data-anchor-id="vocabularies"><span class="header-section-number">5.3.4</span> Vocabularies</h3>
<p>Each vocabulary has a short case-sensitive unique alphanumeric ID, which generally follows the abbreviated name of the vocabulary, omitting dashes. For example, ICD-9-CM has the vocabulary ID “ICD9CM”. As of 2025, over 140 vocabularies are available through ATHENA and follow different cadence of updates. The source and the version of the vocabularies is defined in the VOCABULARY reference file and documentation for individual vocabularies can be found on GitHub (4,16).</p>
</section>
<section id="concept-classes" class="level3" data-number="5.3.5">
<h3 data-number="5.3.5" class="anchored" data-anchor-id="concept-classes"><span class="header-section-number">5.3.5</span> Concept Classes</h3>
<p>Some vocabularies classify their codes or concepts, denoted through their case-sensitive unique alphanumerical IDs. For example, SNOMED has 33 such concept classes, which SNOMED refers to as “semantic tags”: clinical finding, social context, body structure, etc. These are vertical divisions of the concepts. Others, such as MedDRA or RxNorm, have concept classes classifying horizontal levels in their stratified hierarchies. Vocabularies without any concept classes, such as HCPCS, use the vocabulary ID as the Concept Class ID.</p>
<div id="tbl-part-chapter-shorttableterm" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5.1: Vocabularies with or without horizontal and vertical sub-classification principles in concept class.
</figcaption>
<div aria-describedby="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 26%">
<col style="width: 73%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Concept class subdivision principle</th>
<th style="text-align: left;">Vocabulary</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Horizontal</td>
<td style="text-align: left;">All drug vocabularies, CDТ, Episode, HCPCS, HemOnc, ICDs, MedDRA, OSM, Census</td>
</tr>
<tr class="even">
<td style="text-align: left;">Vertical</td>
<td style="text-align: left;">CIEL, HES Specialty, ICDO3, MeSH, NAACCR, NDFRT, OPCS4, PCORNET, Plan, PPI, Provider, SNOMED, SPL, UCUM</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mixed</td>
<td style="text-align: left;">CPT4, ISBT, LOINC</td>
</tr>
<tr class="even">
<td style="text-align: left;">None</td>
<td style="text-align: left;">OXMIS, Race, Revenue Code, Sponsor, Supplier, UB04s, Visit</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>Horizontal concept classes allow you to determine a specific hierarchical level. For example, in the drug vocabulary RxNorm, the concept class “Ingredient” defines the top level of the hierarchy. In the vertical model, members of a concept class can be of any hierarchical level from the top to the very bottom. Concept class is mostly a descriptive attribute and helps to filter concepts. For example, if you only want to select drugs with a specific Brand Name you can filter to “Branded Drug” class.</p>
</section>
<section id="standard-concepts" class="level3" data-number="5.3.6">
<h3 data-number="5.3.6" class="anchored" data-anchor-id="standard-concepts"><span class="header-section-number">5.3.6</span> Standard Concepts</h3>
<p>A Standard Concept is the community-endorsed, canonical representation of a clinical meaning within the OHDSI Vocabularies. It serves as the unified semantic identifier for a specific entity (for example, condition, drug, procedure), regardless of how that entity is expressed in source vocabularies. Only Standard Concepts are used to populate the <code>CONCEPT_ID</code> fields in the CDM, ensuring consistency across diverse datasets. Standard concepts serve as the target for mappings. For each clinical entity, one concept from one vocabulary is chosen to be standard. This becomes the “hub” to which all equivalent source codes are mapped. For example, MESH code D001281, CIEL code 148203, SNOMED code 49436004, ICD9CM code 427.31 and Read code G573000 all define “Atrial fibrillation” in the condition domain, but only the SNOMED concept is Standard and represents the condition in the data. The others are designated non-standard or source concepts and mapped to the Standard ones. Standard Concepts are indicated through an “S” in the <code>STANDARD_CONCEPT</code> field. And only these Standard Concepts are used to record data in the CDM fields ending in <code>_CONCEPT_ID</code>.</p>
<p>We rely on well-known reference terminologies for standard terms: SNOMED CT for conditions, RxNorm and RxNorm Extension for drugs, LOINC and SNOMED for measurements, etc. Not all concepts in those vocabularies are necessarily standard. Occasionally, a concept in a standard vocabulary might be deemed out of scope or duplicative and not used. Conversely, some concepts from typically non-standard vocabularies might be made standard if no better alternative exists.</p>
<p>While we strive to align with the unique standard concept principle to have one Standard concept per semantic entity, duplicates exist. For example, no deduplication of standard concepts has been performed for the Device domain. While concept mappings avoid direct collisions, this dual-standard condition can introduce ambiguity in cohort definition, concept set construction, and analytic interpretation. This phenomenon is not a flaw but rather a reflection of ontology convergence in progress, where two high-quality terminologies independently arrive at comparable representations of the same clinical reality. OHDSI addresses these cases through community review, classification logic, and long-term efforts toward consolidation via concept deprecation, reclassification, or updated mappings. Until then, such duplications must be handled with care in concept set design and ETL strategies.</p>
</section>
<section id="non-standard-concepts" class="level3" data-number="5.3.7">
<h3 data-number="5.3.7" class="anchored" data-anchor-id="non-standard-concepts"><span class="header-section-number">5.3.7</span> Non-Standard Concepts</h3>
<p>Non-standard concepts are not used in standardized analytics, but they are still part of the Standardized Vocabularies and are often found in the source data. For that reason, they are also called “source concepts”. The conversion of source concepts to Standard Concepts is a process called “mapping”.</p>
<p>Some of the non-standard concepts cannot be mapped and are not suitable for analytic use. Examples of such include terms like “Not reported”, “Not specified”, “Passport number” and more.</p>
</section>
<section id="classification-concepts" class="level3" data-number="5.3.8">
<h3 data-number="5.3.8" class="anchored" data-anchor-id="classification-concepts"><span class="header-section-number">5.3.8</span> Classification Concepts</h3>
<p>These concepts are not Standard and hence cannot be used to represent the data. But they are participating in the hierarchy with the Standard Concepts and can therefore be used to perform hierarchical queries. For example, querying for all descendants of ATC code prednisolone;systemic will retrieve the Standard RxNorm concept for prednisolone 5 MG Oral Tablet (<a href="#fig-representation-vocabularies-030-standard-hierarchy" class="quarto-xref">Figure&nbsp;<span>5.3</span></a>).</p>
<div class="landscape">
<div id="fig-representation-vocabularies-030-standard-hierarchy" class="quarto-float quarto-figure quarto-figure-center anchored" alt="standard-hierarchy">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-representation-vocabularies-030-standard-hierarchy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/vocabularies/fig-representation-vocabularies-030-standard-hierarchy.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;5.3: Standard, non-standard source and classification concepts and their hierarchical relationships in the drug domain."><img src="images/vocabularies/fig-representation-vocabularies-030-standard-hierarchy.png" class="img-fluid figure-img" alt="standard-hierarchy"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-representation-vocabularies-030-standard-hierarchy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.3: Standard, non-standard source and classification concepts and their hierarchical relationships in the drug domain.
</figcaption>
</figure>
</div>
</div>
<p>Classification concepts are marked with a “C” in the <code>STANDARD_CONCEPT</code> field. Most classification concepts form a hierarchy along with the standard concepts, and these relationships are stored in the <code>CONCEPT_ANCESTOR</code> table.</p>
<p>Classification concepts are vital in enabling concept set expansion via ancestry traversal. While they cannot be used to populate clinical event tables directly, they serve as entry points into clinically meaningful groupings (for example, drug classes or disorder categories). They are especially powerful when used in cohort definitions or phenotype algorithms that require aggregation of clinically related Standard Concepts. For example, selecting the ATC classification concept C09AA ACE inhibitors will retrieve all Standard RxNorm ingredients and products mapped as descendants.</p>
<p>Quality and coverage of classification hierarchies vary across domains. Currently, Drug and Condition domains have mature classification structures (for example, ATC, MedDRA), while Procedure, Measurement, and Device domains lack formal classification vocabularies. Caution should be exercised when interpreting classification-derived hierarchies, as they may not always reflect clinical practice or data granularity.</p>
<p>5.2.8.1 Standard/non-standard/classification Concept Designation</p>
<p>The choice of concept designation as Standard, non-standard, and classification is typically done for each domain separately at the vocabulary level. This is based on the quality of the concepts, the built-in hierarchy, and the declared purpose of the vocabulary. Also, not all concepts in a vocabulary are used as Standard Concepts. The designation is separate for each domain, each concept must be active, and there might be an order of precedence if more than one concept from different vocabularies compete for the same meaning. See Table&nbsp;5.2&nbsp;for examples.</p>
<div class="landscape">
<div id="tbl-part-chapter-shorttableterm" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5.2: List of vocabularies to utilize for Standard/non-standard/classification concept assignments.
</figcaption>
<div aria-describedby="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 9%">
<col style="width: 33%">
<col style="width: 36%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Domain</th>
<th style="text-align: left;">for Standard Concepts</th>
<th style="text-align: left;">for source concepts</th>
<th style="text-align: left;">for classification concepts</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Condition</td>
<td style="text-align: left;">SNOMED, ICDO3</td>
<td style="text-align: left;">SNOMED Veterinary</td>
<td style="text-align: left;">MedDRA</td>
</tr>
<tr class="even">
<td style="text-align: left;">Procedure</td>
<td style="text-align: left;">SNOMED, CPT4, HCPCS, ICD10PCS, ICD9Proc, OPCS4</td>
<td style="text-align: left;">SNOMED Veterinary, HemOnc, NAACCR</td>
<td style="text-align: left;">None at this point</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Measurement</td>
<td style="text-align: left;">SNOMED, LOINC</td>
<td style="text-align: left;">SNOMED Veterinary, NAACCR, CPT4, HCPCS, OPCS4, PPI</td>
<td style="text-align: left;">None at this point</td>
</tr>
<tr class="even">
<td style="text-align: left;">Drug</td>
<td style="text-align: left;">RxNorm, RxNorm Extension, CVX</td>
<td style="text-align: left;">HCPCS, CPT4, HemOnc, NAACCR</td>
<td style="text-align: left;">ATC</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Device</td>
<td style="text-align: left;">SNOMED</td>
<td style="text-align: left;">Others, currently not normalized</td>
<td style="text-align: left;">None at this point</td>
</tr>
<tr class="even">
<td style="text-align: left;">Observation</td>
<td style="text-align: left;">SNOMED</td>
<td style="text-align: left;">Others</td>
<td style="text-align: left;">None at this point</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Visit</td>
<td style="text-align: left;">CMS Place of Service, ABMT, NUCC</td>
<td style="text-align: left;">SNOMED, HCPCS, CPT4, UB04</td>
<td style="text-align: left;">None at this point</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</div>
</section>
<section id="concept-codes" class="level3" data-number="5.3.9">
<h3 data-number="5.3.9" class="anchored" data-anchor-id="concept-codes"><span class="header-section-number">5.3.9</span> Concept Codes</h3>
<p>Concept codes are the identifiers used in the source vocabularies. For example, ICD9CM or NDC codes are stored in this field, while the OMOP tables use the concept ID as a foreign key into the <code>CONCEPT</code> table. The reason is that the name space overlaps across vocabularies, that is&nbsp;the same code can exist in different vocabularies with completely different meanings (Table&nbsp;5.3).</p>
<div class="landscape">
<div id="tbl-part-chapter-shorttableterm" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5.3: Concepts with identical concept code 1001, but different vocabularies, domains and concept classes.
</figcaption>
<div aria-describedby="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 9%">
<col style="width: 11%">
<col style="width: 43%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 13%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: right;">Concept ID</th>
<th style="text-align: right;">Concept Code</th>
<th style="text-align: left;">Concept<br>Name</th>
<th style="text-align: left;">Domain<br>ID</th>
<th style="text-align: left;">Vocabulary<br>ID</th>
<th style="text-align: left;">Concept<br>Class</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: right;">35803438</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Granulocyte colony-stimulating factors</td>
<td style="text-align: left;">Drug</td>
<td style="text-align: left;">HemOnc</td>
<td style="text-align: left;">Component Class</td>
</tr>
<tr class="even">
<td style="text-align: right;">35942070</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">AJCC TNM Clin T</td>
<td style="text-align: left;">Measurement</td>
<td style="text-align: left;">NAACCR</td>
<td style="text-align: left;">NAACCR Variable</td>
</tr>
<tr class="odd">
<td style="text-align: right;">1036059</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Antipyrine</td>
<td style="text-align: left;">Drug</td>
<td style="text-align: left;">RxNorm</td>
<td style="text-align: left;">Ingredient</td>
</tr>
<tr class="even">
<td style="text-align: right;">38003544</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Residential Treatment - Psychiatric</td>
<td style="text-align: left;">Revenue Code</td>
<td style="text-align: left;">Revenue Code</td>
<td style="text-align: left;">Revenue Code</td>
</tr>
<tr class="odd">
<td style="text-align: right;">43228317</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Aceprometazine maleate</td>
<td style="text-align: left;">Drug</td>
<td style="text-align: left;">BDPM</td>
<td style="text-align: left;">Ingredient</td>
</tr>
<tr class="even">
<td style="text-align: right;">45417187</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Brompheniramine Maleate, 10 mg/mL injectable solution</td>
<td style="text-align: left;">Drug</td>
<td style="text-align: left;">Multum</td>
<td style="text-align: left;">Multum</td>
</tr>
<tr class="odd">
<td style="text-align: right;">45912144</td>
<td style="text-align: right;">1001</td>
<td style="text-align: left;">Serum</td>
<td style="text-align: left;">Specimen</td>
<td style="text-align: left;">CIEL</td>
<td style="text-align: left;">Specimen</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</div>
<p><code>CONCEPT_CODE</code> is unique only within a given vocabulary. You should not join datasets via <code>CONCEPT_CODE</code> unless constrained by <code>VOCABULARY_ID</code>.</p>
<p>In addition, certain vocabularies, such as HCPCS, NDC, and DRG are known to reuse codes over time, assigning new meanings to previously used codes. In such cases, Vocabularies differentiate concepts based on validity dates (<code>VALID_START_DATE</code>, <code>VALID_END_DATE</code>) and keep the most recent meaning.</p>
<p>Some OMOP-specific vocabularies (for example, Type Concept, Visit) contain system-generated concept codes rather than real-world codes. Finally, certain source vocabularies (such as ATC or hierarchical clinical classifications) embed structural hierarchy into their codes (ATC G03E vs.&nbsp;G03EK), meaning that not all <code>CONCEPT_CODE</code> matches imply equivalence at the clinical level.</p>
</section>
<section id="lifecycle" class="level3" data-number="5.3.10">
<h3 data-number="5.3.10" class="anchored" data-anchor-id="lifecycle"><span class="header-section-number">5.3.10</span> Lifecycle</h3>
<p>Vocabularies are rarely permanent corpora with a fixed set of codes. Instead, codes and concepts are added and get deprecated. The OMOP CDM is a model to support longitudinal patient data, which means it needs to support concepts that were used in the past and might no longer be active, as well as supporting new concepts and placing them into context. There are three fields in the <code>CONCEPT</code> table that describe the possible life-cycle statuses: <code>VALID_START_DATE</code>, <code>VALID_END_DATE</code>, and <code>INVALID_REASON</code>. Their values differ depending on the concept life-cycle status:</p>
<ul>
<li><p><strong>Active or new concept</strong></p>
<ul>
<li><p>Description: Concept in use.</p></li>
<li><p><code>VALID_START_DATE</code>: Day of instantiation of concept; if that is not known, day of incorporation of concept in Vocabularies; if that is not known, 1970-1-1.</p></li>
<li><p><code>VALID_END_DATE</code>: Set to 2099-12-31 as a convention to indicate “Might become invalid in an undefined future, but active right now”.</p></li>
<li><p><code>INVALID_REASON</code>: NULL</p></li>
</ul></li>
<li><p><strong>Deprecated Concept with no successor</strong></p>
<ul>
<li><p>Description: Concept inactive and cannot be used as Standard.</p></li>
<li><p><code>VALID_START_DATE</code>: Day of instantiation of concept; if that is not known, day of incorporation of concept in Vocabularies; if that is not known, 1970-1-1.</p></li>
<li><p><code>VALID_END_DATE</code>: Day in the past indicating deprecation, or if that is not known, day of vocabulary refresh where concept in vocabulary went missing or set to inactive.</p></li>
<li><p><code>INVALID_REASON</code>: “D”</p></li>
</ul></li>
<li><p><strong>Upgraded Concept with successor</strong></p>
<ul>
<li><p>Description: Concept inactive but has defined successor. These are typically concepts which went through de-duplication.</p></li>
<li><p><code>VALID_START_DATE</code>: Day of instantiation of concept; if that is not known, day of incorporation of concept in Vocabularies; if that is not known, 1970-1-1.</p></li>
<li><p><code>VALID_END_DATE</code>: Day in the past indicating an upgrade, or if that is not known day of vocabulary refresh where the upgrade was included.</p></li>
<li><p><code>INVALID_REASON</code>: “U”</p></li>
</ul></li>
<li><p><strong>Reused code for another new concept</strong></p>
<ul>
<li><p>Description: The vocabulary reused the concept code of this deprecated concept for a new concept.</p></li>
<li><p><code>VALID_START_DATE</code>: Day of instantiation of concept; if that is not known, day of incorporation of concept in Vocabularies; if that is not known, 1970-1-1.</p></li>
<li><p><code>VALID_END_DATE</code>: Day in the past indicating deprecation, or if that is not known day of vocabulary refresh where concept in vocabulary went missing or set to inactive.</p></li>
</ul></li>
</ul>
<p>In addition to concept lifecycle management, the <code>CONCEPT_RELATIONSHIP</code> table also has lifecycle fields (<code>VALID_START_DATE</code>, <code>VALID_END_DATE</code>, <code>INVALID_REASON</code>) for relationships. Relationships may change over time independently of the concepts themselves. While all relationships are versioned in the internal vocabulary system, only active mappings are included in Athena downloads. Every OMOP CDM instance should record the vocabulary version (stored in the <code>VOCABULARY</code> table) used at ETL time to ensure transparency and reproducibility. Lifecycle management principles apply equally to custom extensions and community-contributed vocabularies: all new concepts and mappings must carry valid <code>VALID_START_DATE</code> entries and, when deprecated, clearly marked <code>VALID_END_DATE</code> and <code>INVALID_REASON</code> values.</p>
</section>
<section id="relationships" class="level3" data-number="5.3.11">
<h3 data-number="5.3.11" class="anchored" data-anchor-id="relationships"><span class="header-section-number">5.3.11</span> Relationships</h3>
<p>Any two concepts can have a defined relationship, regardless of whether the two concepts belong to the same domain or vocabulary. The nature of the relationships is indicated in its short case-sensitive unique alphanumeric ID in the <code>RELATIONSHIP_ID</code> field of the <code>CONCEPT_RELATIONSHIP</code> table. Relationships are symmetrical, that is&nbsp;for each relationship an equivalent relationship exists, where the content of the fields <code>CONCEPT_ID_1</code> and <code>CONCEPT_ID_2</code> are swapped, and the <code>RELATIONHSIP_ID</code> is changed to its opposite. For example, the “Maps to” relationship has an opposite relationship “Mapped from.” Different types of relationships serve different analytic purposes. “Maps to” and “Mapped from” support source-to-standard mappings. “Is a” and “Subsumes” define hierarchical subclass relationships. “Has ingredient” and “Ingredient of” structure drug compositions. “Concept replaced by” and “Concept replaces” handle lifecycle transitions across deprecated content.</p>
<p>As stated in the previous section, <code>CONCEPT_RELATIONSHIP</code> table records also have life-cycle fields <code>VALID_START_DATE</code>, <code>VALID_END_DATE</code> and <code>INVALID_REASON</code>. However, only active records with <code>INVALID_REASON = NULL</code> are available through ATHENA. Inactive relationships are kept for internal processing only.</p>
<p>The <code>RELATIONSHIP</code> table serves as the reference with the full list of relationship IDs and their reverse counterparts. It also specifies two important flags: <code>DEFINES_ANCESTRY</code>, indicating whether a relationship should contribute to the <code>CONCEPT_ANCESTOR</code> table, and <code>IS_HIERARCHICAL</code>, indicating whether the relationship encodes a subsumption hierarchy. Not all relationships define ancestry; only those intended to build domain hierarchies (for example, “Is a”) are used to populate <code>CONCEPT_ANCESTOR</code>. It is essential to distinguish between direct relationships (stored in <code>CONCEPT_RELATIONSHIP</code>) and inferred multi-level hierarchies (precomputed and stored in <code>CONCEPT_ANCESTOR</code>), especially when writing concept set queries, building phenotypes, or exploring ontology structures.</p>
</section>
<section id="mapping-relationships" class="level3" data-number="5.3.12">
<h3 data-number="5.3.12" class="anchored" data-anchor-id="mapping-relationships"><span class="header-section-number">5.3.12</span> Mapping Relationships</h3>
<p>These relationships provide translations from non-standard to Standard concepts, supported by two relationship ID pairs (Table&nbsp;5.4).</p>
<div id="tbl-part-chapter-shorttableterm" class="quarto-float quarto-figure quarto-figure-center anchored" data-tbl-colwidths="[30,70]">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5.4: Type of mapping relationships.
</figcaption>
<div aria-describedby="tbl-part-chapter-shorttableterm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 30%">
<col style="width: 70%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Relationship<br>
ID pair</th>
<th style="text-align: left;">Purpose</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">“Maps to”<br>
and<br>
“Mapped from”</td>
<td style="text-align: left;">Mapping to Standard Concepts. Standard Concepts are mapped to themselves, non-standard concepts to Standard Concepts. Most non-standard and all Standard Concepts have this relationship to a Standard Concept. The former are stored in *_SOURCE_CONCEPT_ID, and the latter in the *_CONCEPT_ID fields. Classification concepts are not mapped.</td>
</tr>
<tr class="even">
<td style="text-align: left;">“Maps to value”<br>
and<br>
“Value mapped from”</td>
<td style="text-align: left;">Mapping to a concept that represents a Value to be placed into the VALUE_AS_CONCEPT_ID fields of the MEASUREMENT and OBSERVATION tables.</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>The purpose of these mapping relationships is to allow a crosswalk between equivalent concepts to harmonize how clinical events are represented in the OMOP CDM. This is a main achievement of the Standardized Vocabularies.</p>
<p>“Equivalent concepts” means it carries the same meaning, and, importantly, the hierarchical descendants cover the same semantic space. If an equivalent concept is not available and the concept is not Standard, it is still mapped, but to a slightly broader concept (so-called “up-hill mappings” or semantic subsumption<strong>)</strong>. For example, ICD10CM W61.51 “Bitten by goose” has no equivalent in the SNOMED vocabulary, which is generally used for standard condition concepts. Instead, it is mapped to SNOMED 217716004 “Peck by bird,” losing the context of the bird being a goose. Up-hill mappings are only used if the loss of information is considered irrelevant to standard research use cases.</p>
<p>Some mappings connect a source concept to more than one Standard Concept. For example, ICD9CM 070.43 “Hepatitis E with hepatic coma” is mapped to both SNOMED 235867002 “Acute hepatitis E” as well as SNOMED 72836002 “Hepatic Coma.” The reason for this is that the original source concept is a pre-coordinated combination of two conditions, hepatitis and coma, meaning that a single code simultaneously encodes multiple clinical ideas rather than expressing them separately SNOMED does not have that combination, which results in two records written to the <code>CONDITION_OCCURRENCE</code> table for the single ICD9CM record, one with each mapped Standard Concept.</p>
<p>Relationships “Maps to value” have the purpose of splitting of a value for OMOP CDM tables following an entity-attribute-value (EAV) model (21). This is typically the case in the following situations:</p>
<ul>
<li><p>Measurements consisting of a test and a result value</p></li>
<li><p>Personal or family disease history</p></li>
<li><p>Allergy to substance</p></li>
<li><p>Need for immunization</p></li>
</ul>
<p>In these situations, the source concept is a combination of the attribute (test or history) and the value (test result or disease). The “Maps to” relationship maps this source to the attribute concept, and the “Maps to value” to the value concept. See <a href="#fig-representation-vocabularies-040-one-to-many" class="quarto-xref">Figure&nbsp;<span>5.4</span></a>&nbsp;for an example.</p>
<div id="fig-representation-vocabularies-040-one-to-many" class="quarto-float quarto-figure quarto-figure-center anchored" alt="one-to-many">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-representation-vocabularies-040-one-to-many-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/vocabularies/fig-representation-vocabularies-040-one-to-many.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;5.4: One-to-many mapping between source concept and Standard Concepts. A pre-coordinated concept is split into two concepts, one of which is the attribute (here history of clinical finding) and the other one is the value (peptic ulcer). While “Maps to” relationship will map to concepts of the measurement or observation domains, the “Maps to value” concepts have no domain restriction."><img src="images/vocabularies/fig-representation-vocabularies-040-one-to-many.png" class="img-fluid figure-img" alt="one-to-many"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-representation-vocabularies-040-one-to-many-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.4: One-to-many mapping between source concept and Standard Concepts. A pre-coordinated concept is split into two concepts, one of which is the attribute (here history of clinical finding) and the other one is the value (peptic ulcer). While “Maps to” relationship will map to concepts of the measurement or observation domains, the “Maps to value” concepts have no domain restriction.
</figcaption>
</figure>
</div>
<p>This process represents a form of controlled <strong>post-coordination</strong> within OMOP vocabularies: instead of encoding every possible combination as a new standard concept, the meaning is decomposed into two (or more) standardized elements that together fully represent the clinical event. Together, they enable more flexible, semantically rich, and extensible data modeling. By post-coordinating attribute and value concepts, OHDSI Standardized Vocabularies avoid uncontrolled growth in the number of concepts while still allowing detailed, clinically meaningful data representation and analysis. Analysts must retrieve both the <code>CONCEPT_ID</code> and <code>VALUE_AS_CONCEPT_ID</code> fields together during query building to reconstruct the complete meaning.</p>
<p>Mapping relationships themselves are subject to lifecycle management. Deprecated mappings (mappings with an <code>INVALID_REASON</code> other than NULL) are removed from active ATHENA releases but can impact longitudinal data or historical cohort definitions if not updated. Careful management of mapping versioning is crucial during vocabulary refresh cycles.</p>
<p>When interpreting mappings, users must be aware that not all source-to-standard mappings imply perfect semantic equivalence. Slight loss of detail, context shift, or broader aggregation may occur, particularly in uphill mappings or when representing pre-coordinated concepts. Analysts and ETL designers should validate mappings in critical analytic contexts.</p>
</section>
<section id="hierarchical-relationships-and-hierarchy" class="level3" data-number="5.3.13">
<h3 data-number="5.3.13" class="anchored" data-anchor-id="hierarchical-relationships-and-hierarchy"><span class="header-section-number">5.3.13</span> Hierarchical Relationships and Hierarchy</h3>
<p>Relationships which indicate a hierarchy are defined through the “Is a” - “Subsumes” relationship pair. Hierarchical relationships are defined such that the child concept has all the attributes of the parent concept, plus one or more additional attributes or a more precisely defined attribute. For example, SNOMED 49436004 “Atrial fibrillation” is related to SNOMED 17366009 “Atrial arrhythmia” through a “Is a” relationship. Both concepts have an identical set of attributes except the type of arrhythmia, which is defined as fibrillation in one but not the other. Concepts can have more than one parent and more than one child concept. In this example, SNOMED 49436004 “Atrial fibrillation” is also an “Is a” to SNOMED 40593004 “Fibrillation.”</p>
<p>Within a domain, and in some cases across domains, standard and classification concepts are organized in a hierarchical structure and stored in the <code>CONCEPT_ANCESTOR</code> table. This allows querying and retrieving concepts and all their hierarchical descendants. These descendants have the same attributes as their ancestor, but also additional or more defined ones.</p>
<p>The <code>CONCEPT_ANCESTOR</code> table is built automatically from the <code>CONCEPT_RELATIONSHIP</code> table, traversing all possible concepts connected through hierarchical relationships. These are the “Is a” - “Subsumes” pairs (<a href="#fig-representation-vocabularies-050-ancestor" class="quarto-xref">Figure&nbsp;<span>5.5</span></a>), and other relationships connecting hierarchies across vocabularies (“SNOMED - CPT4 equivalent”, “RxNorm ingredient of”). The choice whether a relationship participates in the hierarchy constructor is defined for each relationship ID by the flag <code>DEFINES_ANCESTRY</code> in the <code>RELATIONSHIP</code> reference table. It is important to note that not all relationships with hierarchical meaning (<code>IS_HIERARCHICAL</code> = 1) are used for ancestry building; only those with <code>DEFINES_ANCESTRY</code> = 1 contribute to <code>CONCEPT_ANCESTOR</code>. Relationships such as “Has FDA approved indication” or “Consists of” are conceptually hierarchical but are excluded from ancestry paths to preserve clinical rigor.</p>
<div id="fig-representation-vocabularies-050-ancestor" class="quarto-float quarto-figure quarto-figure-center anchored" alt="ancestor">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-representation-vocabularies-050-ancestor-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/vocabularies/fig-representation-vocabularies-050-ancestor.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="Figure&nbsp;5.5: Hierarchy of the condition “Atrial fibrillation”. First degree ancestry is defined through “Is a” and “Subsumes” relationships, while all higher degree relations are inferred and stored in the CONCEPT_ANCESTOR table. Each concept is also its own descendant with both levels of separation equal to 0."><img src="images/vocabularies/fig-representation-vocabularies-050-ancestor.png" class="img-fluid figure-img" alt="ancestor"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-representation-vocabularies-050-ancestor-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.5: Hierarchy of the condition “Atrial fibrillation”. First degree ancestry is defined through “Is a” and “Subsumes” relationships, while all higher degree relations are inferred and stored in the <code>CONCEPT_ANCESTOR</code> table. Each concept is also its own descendant with both levels of separation equal to 0.
</figcaption>
</figure>
</div>
<p>The ancestral degree, or the number of steps between ancestor and descendant, is captured in the <code>MIN_LEVELS_OF_SEPARATION</code> and <code>MAX_LEVELS_OF_SEPARATION</code> fields, defining the shortest or longest possible connection. Not all hierarchical relationships contribute equally to the levels-of-separation calculation. A step counted for the degree is determined by the <code>IS_HIERARCHICAL</code> flag in the <code>RELATIONSHIP</code> reference table for each relationship ID.</p>
<p>As of 2025, a high-quality comprehensive hierarchy exists only for two domains: Drug and Condition. Procedure, Measurement, and Observation domains are only partially covered and in the process of construction. The ancestry is particularly useful for the drug domain as it allows browsing all drugs with a given ingredient or members of drug classes irrespective of the country of origin, brand name or other attributes.</p>
<p>Users should also be aware that vocabulary updates can introduce changes to hierarchical structures, as relationships may be added, modified, or deprecated over time. Therefore, researchers are strongly encouraged to version-control their vocabulary snapshot to preserve analytic reproducibility.</p>
</section>
<section id="other-relationships" class="level3" data-number="5.3.14">
<h3 data-number="5.3.14" class="anchored" data-anchor-id="other-relationships"><span class="header-section-number">5.3.14</span> Other Relationships</h3>
<p>Relationships between two different vocabularies other than mapping and hierarchy relationships are typically of the type “Vocabulary A - Vocabulary B equivalent”, which is either supplied by the original source of the vocabulary or is built de-novo. They may serve as approximate mappings but often are less precise than the better curated mapping relationships. High-quality equivalence relationships (such as “Source - RxNorm equivalent”) are always duplicated by “Maps to” relationship.</p>
<p>Internal vocabulary relationships are usually supplied by the vocabulary provider and their quality highly depends on the vocabulary. Many of these define relationships between clinical events and can be used for information retrieval. For example, disorders of the urethra can be found by following the “Finding site of” relationship (Table&nbsp;5.5):</p>
<div id="tbl-understanding-ehr-versus-claims" class="quarto-float quarto-figure quarto-figure-center anchored" data-tbl-colwidths="[45,55]">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-understanding-ehr-versus-claims-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5.5: “Finding site of” relationship of the “Urethra,” indicating conditions that are situated all in this anatomical structure.
</figcaption>
<div aria-describedby="tbl-understanding-ehr-versus-claims-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 45%">
<col style="width: 55%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;"><code>CONCEPT_ID_1</code></th>
<th style="text-align: left;"><code>CONCEPT_ID_2</code></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">4000504 “Urethra part”</td>
<td style="text-align: left;">36713433 “Partial duplication of urethra”</td>
</tr>
<tr class="even">
<td style="text-align: left;">4000504 “Urethra part”</td>
<td style="text-align: left;">433583 “Epispadias”</td>
</tr>
<tr class="odd">
<td style="text-align: left;">4000504 “Urethra part”</td>
<td style="text-align: left;">443533 “Epispadias, male”</td>
</tr>
<tr class="even">
<td style="text-align: left;">4000504 “Urethra part”</td>
<td style="text-align: left;">4005956 “Epispadias, female”</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>Internal relationships within a vocabulary may represent hierarchical (for example, “Is a”, “RxNorm ingredient of”) connections or non-hierarchical semantic associations such as anatomical location, causative agent, or associated morphology. For example, within RxNorm, relationships like “Precise ingredient of” and “Has precise ingredient” enable navigation between drug products and their precise ingredients.</p>
</section>
</section>
<section id="special-situations" class="level2" data-number="5.4">
<h2 data-number="5.4" class="anchored" data-anchor-id="special-situations"><span class="header-section-number">5.4</span> Special Situations</h2>
<section id="device-coding" class="level3" data-number="5.4.1">
<h3 data-number="5.4.1" class="anchored" data-anchor-id="device-coding"><span class="header-section-number">5.4.1</span> Device Coding</h3>
<p>Device concepts have no standardized coding scheme that could be used to source Standard Concepts. In many source data, devices are not even coded or contained in an external coding scheme. For this same reason, there is currently no hierarchical system available. External standards like GMDN and FDA’s UDI database have been considered but are not yet integrated. As a result, device concepts in OHDSI are mostly standard, same devices have multiple standard concepts across different vocabularies and there is no hierarchy to group terms. If you need help with devices or want to contribute talk to the OHDSI Device Workgroup and refer to <strong>{Chapter 7}</strong> of this book.</p>
</section>
<section id="coding-in-oncology" class="level3" data-number="5.4.2">
<h3 data-number="5.4.2" class="anchored" data-anchor-id="coding-in-oncology"><span class="header-section-number">5.4.2</span> Coding in Oncology</h3>
<p>Cancer data present unique modeling challenges due to the complexity of diagnoses, staging, histology, metastasis, genomic features, and treatment pathways. Please refer to the OHDSI Oncology Workgroup to learn more about conventions.</p>
<p>There are several mapping principles we want to cover in this chapter:</p>
<ul>
<li>Primary cancer diagnoses are mapped to Condition domain concepts, mostly to SNOMED CT. ICDO-3 terms are used where SNOMED coverage is insufficient.</li>
</ul>
<p>Tumor staging, grading, and metastasis details are captured using the specialized Cancer Modifier vocabulary, which encodes structured AJCC/UICC-based elements. Mappings in Cancer Modifier are designed to ensure that cancer-related data: (1) preserve key clinical distinctions (for example, metastatic vs. localized disease), (2) support longitudinal cohort definitions (for example, new diagnosis vs. recurrence), (3) enable harmonized analytics across registries, EHRs, and claims data.</p>
<ul>
<li><p>Genomic abnormalities, when available, are mapped to concepts in the OMOP Genomic vocabulary.</p></li>
<li><p>Oncology-specific measurements and observations, such as tumor dimensions or metastasis spread, often use post-coordination approaches - representing the entity and its result separately - to align with OMOP’s Measurement/Observation model.</p></li>
<li><p>Chemotherapy regimens are represented using the HemOnc vocabulary, while individual oncology drugs are mapped via RxNorm/RxNorm Extension.</p></li>
</ul>
<p>More work is needed to refine mappings, remove duplicates, expand support for hematologic malignancies, and integrate molecular/genomic features.</p>
</section>
<section id="coding-in-psychiatry" class="level3" data-number="5.4.3">
<h3 data-number="5.4.3" class="anchored" data-anchor-id="coding-in-psychiatry"><span class="header-section-number">5.4.3</span> Coding in Psychiatry</h3>
<p>Psychiatric and neuropsychiatric data pose unique challenges for standardization due to the complexity of symptoms, variability of assessment tools, and evolving diagnostic frameworks. If you interested in this research talk to the OHDSI Psychiatry Workgroup.</p>
<p>In the OMOP model, psychiatric assessments are primarily captured within the Measurement and Observation domains, depending on whether the recorded information reflects a quantitative value or a qualitative clinical finding. Workgroup works on integrating and harmonizing Neuropsychiatric Assessment Tools, which include standardized psychometric scales, questionnaires, and structured interviews, into the Vocabularies, deduplicating terms and developing a hierarchy based on SNOMED structure to connect measurements to clinical concepts. They consider using Thesaurus of Psychological Index Terms and Human Phenotype Ontology (HPO), and real-world datasets (for example, MIMIC-IV) to inform this integration.</p>
</section>
<section id="coding-for-gis-exposomes-and-sdoh" class="level3" data-number="5.4.4">
<h3 data-number="5.4.4" class="anchored" data-anchor-id="coding-for-gis-exposomes-and-sdoh"><span class="header-section-number">5.4.4</span> Coding for GIS, Exposomes and SDOH</h3>
<p>Environmental context, exposomes, geographic location, and social conditions are not represented well in the OHDSI Vocabularies. If you are interested in research, talk to the OHDSI GIS Workgroup. One of the outputs of group is the OMOP GIS Vocabulary Package, which (22) delivers three coordinated vocabularies: OMOP GIS for geographic units and spatial relations, OMOP Exposome for chemicals, pollutants, toxins, and their biological targets, and OMOP SDOH for structured social-determinant indicators.</p>
<p>To accommodate these concepts, the package adds new domain identifiers such as Geographic Feature, Environmental Feature, Socioeconomic Feature, and Behavioral Feature. Unlike the classical OMOP domains - essentially routing flags that direct ETL to a specific CDM table - these new domains act solely as semantic groupers. They organize concepts into coherent knowledge families without prescribing storage location. Events encoded with these concepts are still recorded in the appropriate CDM tables such as <code>EXTERNAL_EXPOSURE</code>, <code>OBSERVATION</code>, or <code>MEASUREMENT</code> following existing conventions.</p>
</section>
<section id="microbiology-and-susceptibility-coding" class="level3" data-number="5.4.5">
<h3 data-number="5.4.5" class="anchored" data-anchor-id="microbiology-and-susceptibility-coding"><span class="header-section-number">5.4.5</span> Microbiology and Susceptibility Coding</h3>
<p>There are no comprehensive conventions for microbiology coding in OHDSI. You should refer to Themis conventions for the up-to-date guidance. Generally, the most common scenarios involve (1) specimen collection with a single diagnostic result (for example, Gram stain), (2) multiple lab procedures on a single sample, (3) culture-negative findings, and (4) one or more organisms identified and tested against antibiotics.</p>
<p>OMOP CDM supports this complexity through the <code>MEASUREMENT</code>, <code>OBSERVATION</code>, and <code>SPECIMEN</code> tables, with event linkages (*_EVENT_ID) connecting susceptibility results to organisms and organisms to specimens. Antibiotic susceptibility results are typically stored as LOINC-coded MEASUREMENTs with quantitative values (for example, MIC) and qualitative interpretations (for example, sensitive). When coding microbiology data you should use standard concepts from Measurement domain to populate <code>MEASUREMENT_CONCEPT_ID</code> (such as susceptibility test) and Meas Value domain to populate <code>VALUE_AS_CONCEPT_ID</code> (such as detected/not detected).</p>
</section>
<section id="survey-coding" class="level3" data-number="5.4.6">
<h3 data-number="5.4.6" class="anchored" data-anchor-id="survey-coding"><span class="header-section-number">5.4.6</span> Survey Coding</h3>
<p>There are no comprehensive conventions for survey coding in OHDSI. You should refer to Survey Workgroup for the up-to-date guidance. Broadly, surveys can be stored as Question-Answer pairs (separate concepts) or as pre-coordinated Question-Answer (one concept). Existing survey vocabularies, such as PPI and UK Biobank, are a mix of both. Surveys added to the Vocabularies generally should follow broad Vocabularies principles. For example, they should not contain negative information and flavors of null (not reported, not specified, etc.). If they have codes that already have standard counterparts in the Vocabularies, they should be mapped appropriately. If you want to add your survey instrument, please talk to the Survey Workgroup.</p>
</section>
<section id="flavors-of-null" class="level3" data-number="5.4.7">
<h3 data-number="5.4.7" class="anchored" data-anchor-id="flavors-of-null"><span class="header-section-number">5.4.7</span> Flavors of NULL</h3>
<p>Many vocabularies contain codes that represent some form of absence of information. For example, of the five gender concepts 8507 “Male,” 8532 “Female,” 8570 “Ambiguous,” 8551 “Unknown,” and 8521 “Other”, only the first two are Standard, and the other three are source concepts with no mapping. In the Standardized Vocabularies, there is intentionally no distinction why a piece of information is not available; it might be because of an active withdrawal of information by the patient, a missing value, a value that is not defined or standardized in some way, or the absence of a mapping record in <code>CONCEPT_RELATIONSHIP</code>. Any such concept is not mapped, which corresponds to a default mapping to the Standard Concept with the concept ID = 0.</p>
<p>As per Vocabularies’ principles we avoid adding new flavors of NULL to the Vocabularies and advise against using such concepts in research.</p>
</section>
</section>
<section id="summary" class="level2" data-number="5.5">
<h2 data-number="5.5" class="anchored" data-anchor-id="summary"><span class="header-section-number">5.5</span> Summary</h2>
<ul>
<li><p>All events and administrative facts are represented in the OHDSI Standardized Vocabularies as concepts and concept relationships.</p></li>
<li><p>Most of these are adopted from existing coding schemes or vocabularies, while others are either extended (for example, RxNorm Extension, OMOP Extension) or developed de novo by OHDSI Vocabulary Team or community to cover missing areas.</p></li>
<li><p>All concepts are assigned a domain, which controls where the fact represented by the concept is stored in the CDM.</p></li>
<li><p>Concepts of equivalent meaning in different vocabularies are mapped to one of them, which is designated the Standard Concept. The others are source concepts. Standard concepts (“S”) are the only concepts used in analytical fields.</p></li>
<li><p>We strive for collaborative and transparent Vocabularies with most of the documentation located on OHDSI Vocabularies GitHub Wiki. You can get involved as a community contributor or vocabulary steward. You can contribute simple content through templates or more complex content though programmatic vocabulary development.</p></li>
</ul>
<p><strong>References</strong></p>
<p>1. Reich C, Ostropolets A, Ryan P, Rijnbeek P, Schuemie M, Davydov A, et al.&nbsp;OHDSI Standardized Vocabularies-a large-scale centralized reference ontology for international data harmonization. J Am Med Inform Assoc. 2024 Feb 16;31(3):583–90.</p>
<p>2. Athena [Internet]. [cited 2025 May 23]. Available from: https://athena.ohdsi.org/search-terms/start</p>
<p>3. Release planning · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 23]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Release-planning</p>
<p>4. Standardized Vocabularies · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 23]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Standardized-Vocabularies</p>
<p>5. Community contribution · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 23]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution</p>
<p>6. Dymshyts D. Evaluating the impact of different vocabulary versions on cohort definitions and CDM. In 2024 [cited 2025 May 23]. Available from: https://www.ohdsi.org/wp-content/uploads/2024/10/23-EvaluationConceptSets_Ddymshyts_2024_US-Dmitry-Dymshyts.pdf</p>
<p>7. Amos L, Anderson D, Brody S, Ripple A, Humphreys BL. UMLS users and uses: a current overview. Journal of the American Medical Informatics Association. 2020 Oct 1;27(10):1606–11.</p>
<p>8. De Groot R, Glaser S, Kogan A, Medlock S, Alloni A, Gabetta M, et al.&nbsp;ATC-to-RxNorm mappings – A comparison between OHDSI Standardized Vocabularies and UMLS Metathesaurus. International Journal of Medical Informatics. 2025 Mar;195:105777.</p>
<p>9. A High-Fidelity Combined ATC-Rxnorm Drug Hierarchy for Large-Scale Observational Research. In: Studies in Health Technology and Informatics [Internet]. IOS Press; 2024 [cited 2025 May 23]. Available from: https://ebooks.iospress.nl/doi/10.3233/SHTI230926</p>
<p>10. General Structure, Download and Use · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 23]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/General-Structure,-Download-and-Use</p>
<p>11. Trofymenko M, Talapova P, Williams A. OMOP GIS Vocabulary Package for Observational Studies in Health Care and Public Health. In.</p>
<p>12. GitHub [Internet]. [cited 2025 May 26]. Vocabulary Development Process. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Vocabulary-Development-Process</p>
<p>13. Quality Assurance and Control · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 26]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Quality-assurance-and-control</p>
<p>14. Introduction · OHDSI/Vocabulary-v5.0 Wiki · GitHub [Internet]. [cited 2025 May 26]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Introduction</p>
<p>15. Ostropolets A. OHDSI Vocabularies landscape assessment [Internet]. 2023. Available from: https://ohdsiorg.sharepoint.com/:w:/s/Workgroup-CommonDataModel/EQZxds1n62JIsywmDCknwtABnSb42q7hM5PwiyblXV9zDw?e=cyvxi0</p>
<p>16. Releases · OHDSI/Vocabulary-v5.0 [Internet]. [cited 2025 May 23]. Available from: https://github.com/OHDSI/Vocabulary-v5.0/releases</p>
<p>17. Matentzoglu N, Balhoff JP, Bello SM, Bizon C, Brush M, Callahan TJ, et al.&nbsp;A Simple Standard for Sharing Ontological Mappings (SSSOM). Database. 2022 May 25;2022:baac035.</p>
<p>18. OHDSI/Tantalus [Internet]. Observational Health Data Sciences and Informatics; 2024 [cited 2025 May 28]. Available from: https://github.com/OHDSI/Tantalus</p>
<p>19. Park Y, Yoon J, Zhuk A, Ostropolets A, You SC. Integrating Local Vocabulary into OMOP CDM: A Step-by-Step Tutorial [Internet]. 2025 [cited 2025 May 26]. Available from: http://medrxiv.org/lookup/doi/10.1101/2025.05.07.25327200</p>
<p>20. GitHub [Internet]. [cited 2025 May 23]. Domains. Available from: https://github.com/OHDSI/Vocabulary-v5.0/wiki/Domains</p>
<p>21. Dinu V, Nadkarni P. Guidelines for the effective use of entity–attribute–value modeling for biomedical databases. International Journal of Medical Informatics. 2007 Nov;76(11–12):769–79.</p>
<p>22. OHDSI GIS WG [Internet]. 2025 [cited 2025 May 28]. Available from: https://ohdsi.github.io/GIS/vocabulary.html</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../representation/cdm.html" class="pagination-link" aria-label="The Common Data Model">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">The Common Data Model</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../representation/etl.html" class="pagination-link" aria-label="Extract Transform Load">
        <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Extract Transform Load</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p><a href="https://ohdsi.org/">OHDSI</a></p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    <div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/edit/main/representation/vocabularies.md" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li><li><a href="https://github.com/OHDSI/BookOfOhdsi-2ndEdition/blob/main/representation/vocabularies.md" class="toc-action"><i class="bi empty"></i>View source</a></li></ul></div></div>
    <div class="nav-footer-right">
<p>© <a href="https://creativecommons.org/licenses/by-nd/4.0/"> CC BY-ND 4</a></p>
</div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>